Index: branches/2.0/postgis/lwgeom_accum.c
===================================================================
--- postgis/lwgeom_accum.c.old
+++ postgis/lwgeom_accum.c
@@ -106,18 +106,17 @@
 		         errmsg("could not determine input data type")));
 
+#if POSTGIS_PGSQL_VERSION == 84
 	if (fcinfo->context && IsA(fcinfo->context, AggState))
+	{
 		aggcontext = ((AggState *) fcinfo->context)->aggcontext;
-#if POSTGIS_PGSQL_VERSION == 84
-
+	}
 	else if (fcinfo->context && IsA(fcinfo->context, WindowAggState))
+	{
 		aggcontext = ((WindowAggState *) fcinfo->context)->wincontext;
+	}
+	else
+#else
+	if ( ! AggCheckCallContext(fcinfo, &aggcontext) )
 #endif
-#if POSTGIS_PGSQL_VERSION > 84
-
-	else if (fcinfo->context && IsA(fcinfo->context, WindowAggState))
-		aggcontext = ((WindowAggState *) fcinfo->context)->aggcontext;
-#endif
-
-	else
 	{
 		/* cannot be called directly because of dummy-type argument */
