--- pkg/pgadmin3.desktop.old	2016-02-17 22:30:57.592681020 +0200
+++ pkg/pgadmin3.desktop	2016-02-17 22:31:23.904431693 +0200
@@ -1,11 +1,11 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=pgAdmin III
-Exec=/usr/bin/pgadmin3
-Icon=/usr/share/pgadmin3/pgAdmin3.png
+Exec=/usr/bin/pgadmin3_95
+Icon=/usr/share/pgadmin3_95/pgadmin3_95.xpm
 Type=Application
 Categories=Application;Development;
 MimeType=text/html
-DocPath=/usr/share/pgadmin3/docs/en_US/index.html
+DocPath=//usr/pgsql-9.5/share/pgadmin3/docs/en_US/index.html
 Comment=PostgreSQL Tools
 Keywords=database;db;sql;query;administration;development;
