diff -up plruby-0.5.4/extconf.rb.version plruby-0.5.4/extconf.rb
--- plruby-0.5.4/extconf.rb.version	2014-04-28 10:32:23.290943486 +0200
+++ plruby-0.5.4/extconf.rb	2014-04-28 10:32:59.968953807 +0200
@@ -117,7 +117,7 @@ if have_func("rb_hash_delete", "ruby.h")
 end
 
 case version_str = `#{pg_config} --version`
-when /^PostgreSQL ([7-9])\.([0-9])(\.[0-9])?$/
+when /^PostgreSQL ([7-9])\.([0-9]{1,3})(\.[0-9]{1,3})?$/
    version = 10 * $1.to_i + $2.to_i
 else
    version = 0
