--- Makefile.old	2020-08-06 23:05:28.260287647 +0100
+++ Makefile	2020-08-06 23:05:53.141546250 +0100
@@ -5,12 +5,12 @@
 DATA = tdigest--1.0.0.sql
 MODULES = tdigest
 
-CFLAGS=`pg_config --includedir-server`
+CFLAGS=`/usr/pgsql-11/bin/pg_config --includedir-server`
 
 TESTS        = $(wildcard test/sql/*.sql)
 REGRESS      = $(patsubst test/sql/%.sql,%,$(TESTS))
 REGRESS_OPTS = --inputdir=test
 
-PG_CONFIG = pg_config
+PG_CONFIG = /usr/pgsql-11/bin/pg_config
 PGXS := $(shell $(PG_CONFIG) --pgxs)
 include $(PGXS)
