--- Makefile.old	2020-11-12 00:42:40.324254641 +0000
+++ Makefile	2020-11-12 00:43:01.085474672 +0000
@@ -5,12 +5,12 @@
 DATA = tdigest--1.0.0.sql tdigest--1.0.0--1.0.1.sql
 MODULES = tdigest
 
-CFLAGS=`pg_config --includedir-server`
+CFLAGS=`/usr/pgsql-13/bin/pg_config --includedir-server`
 
 TESTS        = $(wildcard test/sql/*.sql)
 REGRESS      = $(patsubst test/sql/%.sql,%,$(TESTS))
 REGRESS_OPTS = --inputdir=test
 
-PG_CONFIG = pg_config
+PG_CONFIG = /usr/pgsql-13/bin/pg_config
 PGXS := $(shell $(PG_CONFIG) --pgxs)
 include $(PGXS)
