--- plsh.c.old	2019-01-20 19:05:13.544974756 +0000
+++ plsh.c	2019-01-20 19:06:29.247409253 +0000
@@ -49,6 +49,9 @@
 #define CALLED_AS_EVENT_TRIGGER(x) 0
 #endif
 
+#if PG_VERSION_NUM < 110000 && !defined(TupleDescAttr)
+#define TupleDescAttr(tupdesc, i) ((tupdesc)->attrs[(i)])
+#endif
 
 static char * handler_internal2(const char *tempfile, char * const * arguments, const char *proname, TriggerData *trigger_data, EventTriggerData *event_trigger_data);
 
@@ -477,10 +480,10 @@
 				if (isnull)
 					s = "";
 				else
-					s = type_to_cstring(attr, tupdesc->attrs[i]->atttypid);
+					s = type_to_cstring(attr, TupleDescAttr(tupdesc, i)->atttypid);
 
 				elog(DEBUG2, "arg %d is \"%s\" (type %u)", i, s,
-					 tupdesc->attrs[i]->atttypid);
+					 TupleDescAttr(tupdesc, i)->atttypid);
 
 				arguments[argc++] = s;
 			}
