--- Makefile.old	2016-03-20 20:56:32.324053882 +0200
+++ Makefile	2016-03-20 20:56:42.118959334 +0200
@@ -2,8 +2,6 @@
 EXTENSION    = $(shell grep -m 1 '"name":' $(META) | sed -e 's/[[:space:]]*"name":[[:space:]]*"\([^"]*\)",/\1/')
 EXTVERSION   = $(shell grep -m 1 '"version":' $(META) | sed -e 's/[[:space:]]*"version":[[:space:]]*"\([^"]*\)",/\1/')
 
-echo 'destdir = $(DESTDIR)'
-    
 SED = sed
 
 SQLSCRIPTS = \
@@ -20,7 +18,7 @@
 # to your extention.
 #
 #MODULES      = $(patsubst %.c,%,$(wildcard src/*.c))
-PG_CONFIG    = pg_config
+PG_CONFIG    = /usr/pgsql-9.6/bin/pg_config
 
 EXTNDIR     = $(shell $(PG_CONFIG) --sharedir)
 PG91         = $(shell $(PG_CONFIG) --version | grep -qE " 8\.| 9\.0" && echo no || echo yes)
