--- Makefile.old	2019-11-03 14:17:44.493353187 +0000
+++ Makefile	2019-11-03 14:18:05.209481316 +0000
@@ -7,12 +7,12 @@
 		sql/count_distinct--2.0.0--3.0.0.sql sql/count_distinct--3.0.0--3.0.1.sql
 MODULES = count_distinct
 
-CFLAGS=`pg_config --includedir-server`
+CFLAGS=`/usr/pgsql-12/bin/pg_config --includedir-server`
 
 TESTS        = $(wildcard test/sql/*.sql)
 REGRESS      = $(patsubst test/sql/%.sql,%,$(TESTS))
 REGRESS_OPTS = --inputdir=test
 
-PG_CONFIG = pg_config
+PG_CONFIG = /usr/pgsql-12/bin/pg_config
 PGXS := $(shell $(PG_CONFIG) --pgxs)
 include $(PGXS)
