--- src/test/perl/Makefile.old	2021-05-21 09:14:54.342287374 +0100
+++ src/test/perl/Makefile	2021-05-21 09:15:25.727334731 +0100
@@ -23,11 +23,13 @@
 	$(INSTALL_DATA) $(srcdir)/SimpleTee.pm '$(DESTDIR)$(pgxsdir)/$(subdir)/SimpleTee.pm'
 	$(INSTALL_DATA) $(srcdir)/RecursiveCopy.pm '$(DESTDIR)$(pgxsdir)/$(subdir)/RecursiveCopy.pm'
 	$(INSTALL_DATA) $(srcdir)/PostgresNode.pm '$(DESTDIR)$(pgxsdir)/$(subdir)/PostgresNode.pm'
+	$(INSTALL_DATA) $(srcdir)/PostgresVersion.pm '$(DESTDIR)$(pgxsdir)/$(subdir)/PostgresVersion.pm'
 
 uninstall:
 	rm -f '$(DESTDIR)$(pgxsdir)/$(subdir)/TestLib.pm'
 	rm -f '$(DESTDIR)$(pgxsdir)/$(subdir)/SimpleTee.pm'
 	rm -f '$(DESTDIR)$(pgxsdir)/$(subdir)/RecursiveCopy.pm'
 	rm -f '$(DESTDIR)$(pgxsdir)/$(subdir)/PostgresNode.pm'
+	rm -f '$(DESTDIR)$(pgxsdir)/$(subdir)/PostgresVersion.pm'
 
 endif
