--- frmts/jpeg2000/jpeg2000_vsil_io.cpp.old	2020-09-08 09:58:40.878188309 +0100
+++ frmts/jpeg2000/jpeg2000_vsil_io.cpp	2020-09-08 09:59:57.667080293 +0100
@@ -96,7 +96,7 @@
 
 // PRIjas_seqent macro is defined since Jasper 2.0.17
 
-#ifndef PRIjas_seqent
+#if 0
 static int JPEG2000_VSIL_read(jas_stream_obj_t *obj, char *buf, int cnt)
 #else
 static int JPEG2000_VSIL_read(jas_stream_obj_t *obj, char *buf, unsigned cnt)
@@ -106,7 +106,7 @@
     return static_cast<int>(VSIFReadL(buf, 1, cnt, fileobj->fp));
 }
 
-#ifndef PRIjas_seqent
+#if 0
 static int JPEG2000_VSIL_write(jas_stream_obj_t *obj, char *buf, int cnt)
 #else
 static int JPEG2000_VSIL_write(jas_stream_obj_t *obj, char *buf, unsigned int cnt)
