--- pyproject.toml.old	2026-01-15 12:47:25.164637746 +0000
+++ pyproject.toml	2026-01-15 12:47:37.848526841 +0000
@@ -4,7 +4,7 @@
 description = "A minimalist extension and utility pair for time series analysis and visualization of PostgreSQL internal statistics."
 readme = "README.md"
 requires-python = ">=3.11"
-license = "PostgreSQL"
+license = {text = "PostgreSQL"}
 keywords = ["open-source", "postgres", "opensource", "database", "dataviz", "time-series", "postgresql", "data-visualization", "database-management", "database-administration", "performance-analysis", "postgresql-database", "postgresql-extension", "time-series-analysis"]
 authors = [{name = "Jimmy Angelakos", email = "vyruss@hellug.gr"}]
 classifiers = ["Development Status :: 4 - Beta",
