diff -up ./pgadmin/db/pgConn.cpp.nullthis ./pgadmin/db/pgConn.cpp
--- ./pgadmin/db/pgConn.cpp.nullthis	2016-02-08 05:19:40.000000000 -0500
+++ ./pgadmin/db/pgConn.cpp	2016-08-28 15:36:47.295256972 -0400
@@ -406,22 +406,19 @@ wxString pgConn::GetSslModeName()
 bool pgConn::GetIsEdb()
 {
 	// to retrieve edb flag
-	BackendMinimumVersion(0, 0);
-	return isEdb;
+	return BackendMinimumVersion(0, 0) && isEdb;
 }
 
 bool pgConn::GetIsGreenplum()
 {
 	// to retrieve Greenplum flag
-	BackendMinimumVersion(0, 0);
-	return isGreenplum;
+	return BackendMinimumVersion(0, 0) && isGreenplum;
 }
 
 bool pgConn::GetIsHawq()
 {
 	// to retrieve Greenplum HAWQ flag
-	BackendMinimumVersion(0, 0);
-	return isHawq;
+	return BackendMinimumVersion(0, 0) && isHawq;
 }
 
 wxString pgConn::SystemNamespaceRestriction(const wxString &nsp)
@@ -475,6 +472,9 @@ bool pgConn::IsSuperuser()
 
 bool pgConn::BackendMinimumVersion(int major, int minor)
 {
+	if (GetStatus() != PGCONN_OK)
+		return false;
+
 	if (!majorVersion)
 	{
 		wxString version = GetVersionString();
@@ -503,8 +503,8 @@ bool pgConn::BackendMinimumVersion(int m
 // releases are not coordinated with PostgreSQL minor releases.
 bool pgConn::BackendMinimumVersion(int major, int minor, int patch)
 {
-	if (!majorVersion)
-		BackendMinimumVersion(0, 0);
+	if (!BackendMinimumVersion(major, minor))
+	  return false; // false if non connected
 
 	return majorVersion > major || (majorVersion == major && minorVersion > minor) || (majorVersion == major && minorVersion == minor && patchVersion >= patch);
 }
@@ -648,6 +648,11 @@ void pgConn::ExamineLibpqVersion()
 	}
 }
 
+int pgConn::GetPort() const
+{
+	return conn ? atoi(PQport(conn)) : 0;
+}
+
 wxString pgConn::GetName() const
 {
 	wxString str;
@@ -1003,15 +1008,15 @@ bool pgConn::IsAlive()
 }
 
 
-int pgConn::GetStatus() const
+int pgConn::GetStatus(const pgConn *p)
 {
-	if (!this)
+	if (!p)
 		return PGCONN_BAD;
 
-	if (conn)
-		((pgConn *)this)->connStatus = PQstatus(conn);
+	if (p->conn)	// FIXME: casting away const !
+		((pgConn *)p)->connStatus = PQstatus(p->conn);
 
-	return connStatus;
+	return p->connStatus;
 }
 
 
diff -up ./pgadmin/include/db/pgConn.h.nullthis ./pgadmin/include/db/pgConn.h
--- ./pgadmin/include/db/pgConn.h.nullthis	2016-02-08 05:19:40.000000000 -0500
+++ ./pgadmin/include/db/pgConn.h	2016-08-23 13:11:14.588564870 -0400
@@ -194,10 +194,7 @@ public:
 	{
 		return utfConnectString;
 	}
-	int GetPort() const
-	{
-		return atoi(PQport(conn));
-	};
+	int GetPort() const;
 	wxString GetTTY() const
 	{
 		return wxString(PQtty(conn), *conv);
@@ -215,7 +212,11 @@ public:
 	{
 		return PQbackendPID(conn);
 	}
-	int GetStatus() const;
+	static int GetStatus(const pgConn *);
+	int GetStatus() const
+	{
+		return GetStatus(this);
+	}
 	int GetLastResultStatus() const
 	{
 		return lastResultStatus;
