--- Makefile.old	2017-06-02 23:05:45.475237800 +0300
+++ Makefile	2017-06-02 23:06:18.403991827 +0300
@@ -9,7 +9,7 @@
 # If working with a PG source directory, point PGSQL_INCLUDE_DIR to its
 # src/include subdirectory.  If working with an installed tree, point to
 # the server include subdirectory, eg /usr/local/include/postgresql/server
-PG_CONFIG=pg_config
+PG_CONFIG=/usr/pgsql-9.6/bin/pg_config
 PGSQL_INCLUDE_DIR=$(shell $(PG_CONFIG) --includedir-server)
 PGSQL_LIB_DIR=$(shell $(PG_CONFIG) --libdir)
 
--- Makefile.contrib.old	2017-06-02 23:05:55.373163864 +0300
+++ Makefile.contrib	2017-06-02 23:06:29.214911072 +0300
@@ -4,7 +4,7 @@
 DOCS = README.pg_filedump
 
 ifdef USE_PGXS
-	PG_CONFIG = pg_config
+	PG_CONFIG = /usr/pgsql-9.6/bin/pg_config
 	PGXS := $(shell $(PG_CONFIG) --pgxs)
 	include $(PGXS)
 else
