--- src/guc.c.old	2019-06-04 17:20:29.860438093 +0100
+++ src/guc.c	2019-06-04 17:20:49.163555144 +0100
@@ -21,7 +21,7 @@
 /* Define which level means on. We use this object to have at least one object
  * of type TelemetryLevel in the code, otherwise pgindent won't work for the
  * type */
-static const TelemetryLevel on_level = TELEMETRY_BASIC;
+static const TelemetryLevel on_level = TELEMETRY_OFF;
 
 bool
 ts_telemetry_on()
@@ -41,7 +41,7 @@
 bool ts_guc_enable_constraint_exclusion = true;
 int ts_guc_max_open_chunks_per_insert = 10;
 int ts_guc_max_cached_chunks_per_hypertable = 10;
-int ts_guc_telemetry_level = TELEMETRY_BASIC;
+int ts_guc_telemetry_level = TELEMETRY_OFF;
 
 TSDLLEXPORT char *ts_guc_license_key = TS_DEFAULT_LICENSE;
 char *ts_last_tune_time = NULL;
@@ -166,7 +166,7 @@
 							 "Telemetry settings level",
 							 "Level used to determine which telemetry to send",
 							 &ts_guc_telemetry_level,
-							 TELEMETRY_BASIC,
+							 TELEMETRY_OFF,
 							 telemetry_level_options,
 							 PGC_USERSET,
 							 0,
--- sql/notice.sql.old	2019-06-04 17:06:00.165164582 +0100
+++ sql/notice.sql	2019-06-04 17:06:34.505372638 +0100
@@ -30,8 +30,7 @@
     ||
     E' 1. Getting started: https://docs.timescale.com/getting-started\n' ||
     E' 2. API reference documentation: https://docs.timescale.com/api\n' ||
-    E' 3. How TimescaleDB is designed: https://docs.timescale.com/introduction/architecture\n\n' ||
-    E'Note: TimescaleDB collects anonymous reports to better understand and assist our users.\nFor more information and how to disable, please see our docs https://docs.timescaledb.com/using-timescaledb/telemetry.',
+    E' 3. How TimescaleDB is designed: https://docs.timescale.com/introduction/architecture ',
     expiration_time_string;
 
 IF now() > end_time
--- sql/updates/0.11.0--0.12.0.sql.old	2019-06-04 17:06:46.296444083 +0100
+++ sql/updates/0.11.0--0.12.0.sql	2019-06-04 17:06:56.764507559 +0100
@@ -51,14 +51,6 @@
 GRANT SELECT ON _timescaledb_config.bgw_job TO PUBLIC;
 GRANT SELECT ON _timescaledb_internal.bgw_job_stat TO PUBLIC;
 
-DO language plpgsql $$
-BEGIN
-  RAISE WARNING '%',
- E'\nStarting in v0.12.0, TimescaleDB collects anonymous reports to better understand and assist our
-users. For more information and how to disable, please see our docs https://docs.timescaledb.com/using-timescaledb/telemetry.\n';
-END;
-$$;
-
 CREATE TABLE IF NOT EXISTS _timescaledb_catalog.installation_metadata (
     key     NAME NOT NULL PRIMARY KEY,
     value   TEXT NOT NULL
