--- repmgr.conf.sample.old	2017-11-21 18:28:38.323665540 +0300
+++ repmgr.conf.sample	2017-11-21 18:30:05.576041447 +0300
@@ -129,7 +129,7 @@
 # Environment/command settings
 #------------------------------------------------------------------------------
 
-#pg_bindir=''                           # Path to PostgreSQL binary directory (location
+pg_bindir='/usr/pgsql-9.5/bin/'         # Path to PostgreSQL binary directory (location
                                         # of pg_ctl, pg_basebackup etc.). Only needed
                                         # if these files are not in the system $PATH.
                                         #
