--- Makefile.old	2016-09-29 00:37:58.775073239 +0300
+++ Makefile	2016-09-29 00:38:22.037869435 +0300
@@ -3,12 +3,14 @@
 CC ?= gcc
 CFLAGS = -g -std=gnu99 -Wall -pedantic
 PREFIX ?= /usr
-PGCONFIG ?= pg_config
+PGCONFIG ?= /usr/pgsql-9.2/bin/pg_config
 PGLIBDIR = $(shell $(PGCONFIG) --libdir)
 PGINCLUDEDIR = $(shell $(PGCONFIG) --includedir)
 ifndef NCONFIG
 	ifeq ($(shell sh -c 'which ncurses5-config>/dev/null 2>/dev/null && echo y'), y)
 		NCONFIG = ncurses5-config
+	else ifeq ($(shell sh -c 'which ncurses6-config>/dev/null 2>/dev/null && echo y'), y)
+		NCONFIG = ncurses6-config
 	else ifeq ($(shell sh -c 'which ncursesw5-config>/dev/null 2>/dev/null && echo y'), y)
 		NCONFIG = ncursesw5-config
 	endif
