--- src/sample/pgpool.conf.sample.old	2016-04-28 10:31:25.843938737 +0300
+++ src/sample/pgpool.conf.sample	2016-04-28 10:31:41.171763055 +0300
@@ -33,8 +33,6 @@
                                    # (change requires restart)
 socket_dir = '/tmp'
                                    # Unix domain socket path
-                                   # The Debian package defaults to
-                                   # /var/run/postgresql
                                    # (change requires restart)
 listen_backlog_multiplier = 2
                                    # Set the backlog parameter of listen(2) to
@@ -55,8 +53,6 @@
                                    # (change requires restart)
 pcp_socket_dir = '/tmp'
                                    # Unix domain socket path for pcp
-                                   # The Debian package defaults to
-                                   # /var/run/postgresql
                                    # (change requires restart)
 
 # - Backend Connection Settings -
@@ -67,7 +63,7 @@
                                    # Port number for backend 0
 backend_weight0 = 1
                                    # Weight for backend 0 (only in load balancing mode)
-backend_data_directory0 = '/var/lib/pgsql/data'
+backend_data_directory0 = '/var/lib/pgsql/9.1/data'
                                    # Data directory for backend 0
 backend_flag0 = 'ALLOW_TO_FAILOVER'
                                    # Controls various backend behavior
@@ -214,10 +210,10 @@
 # FILE LOCATIONS
 #------------------------------------------------------------------------------
 
-pid_file_name = '/var/run/pgpool/pgpool.pid'
+pid_file_name = '/var/run/pgpool-II-91/pgpool.pid'
                                    # PID file name
                                    # (change requires restart)
-logdir = '/var/log/pgpool'
+logdir = '/var/log/pgpool-II-91'
                                    # Directory of pgPool status file
                                    # (change requires restart)
 
