--- Makefile.old	2017-04-28 14:37:01.744966699 +0300
+++ Makefile	2017-04-28 14:37:26.132726324 +0300
@@ -6,7 +6,7 @@
 #  Portions Copyright (c) 2012-2015, The Reorg Development Team
 #
 
-PG_CONFIG ?= pg_config
+PG_CONFIG ?= /usr/pgsql-9.4/bin/pg_config
 EXTENSION = pg_repack
 
 .PHONY: dist/$(EXTENSION)-$(EXTVERSION).zip
--- bin/Makefile.old	2017-04-28 14:37:03.804946395 +0300
+++ bin/Makefile	2017-04-28 14:37:32.672661864 +0300
@@ -6,7 +6,7 @@
 #  Portions Copyright (c) 2012-2015, The Reorg Development Team
 #
 
-PG_CONFIG ?= pg_config
+PG_CONFIG ?= /usr/pgsql-9.4/bin/pg_config
 
 SRCS = pg_repack.c pgut/pgut.c pgut/pgut-fe.c
 OBJS = $(SRCS:.c=.o)
--- regress/Makefile.old	2017-04-28 14:37:06.789916974 +0300
+++ regress/Makefile	2017-04-28 14:37:37.469614584 +0300
@@ -6,7 +6,7 @@
 #  Portions Copyright (c) 2012-2015, The Reorg Development Team
 #
 
-PG_CONFIG ?= pg_config
+PG_CONFIG ?= /usr/pgsql-9.4/bin/pg_config
 
 # version as a number, e.g. 9.1.4 -> 901
 VERSION := $(shell $(PG_CONFIG) --version | awk '{print $$2}')
--- lib/Makefile.old	2017-04-28 14:46:42.468242879 +0300
+++ lib/Makefile	2017-04-28 14:47:06.128009679 +0300
@@ -6,7 +6,7 @@
 #  Portions Copyright (c) 2012-2015, The Reorg Development Team
 #
 
-PG_CONFIG ?= pg_config
+PG_CONFIG ?= /usr/pgsql-9.4/bin/pg_config
 
 # version as a number, e.g. 9.1.4 -> 901
 VERSION := $(shell $(PG_CONFIG) --version | awk '{print $$2}')
