--- setup.py.old	2020-03-02 21:39:04.283859628 +0000
+++ setup.py	2020-03-02 21:39:12.959946650 +0000
@@ -51,7 +51,7 @@
 
 # Get the version string. Cannot be done with import!
 g = {}
-with open(os.path.join('flask_gravatar', 'version.py'), 'rt') as fp:
+with open(os.path.join('flask_gravatar', 'version.py'), 'rt', encoding="utf-8") as fp:
     exec(fp.read(), g)
     version = g['__version__']
 
