--- Makefile.old	2017-09-13 09:07:45.265856473 +0100
+++ Makefile	2017-09-13 09:07:55.400746226 +0100
@@ -31,7 +31,7 @@
 # to your extension.
 #
 #MODULES      = $(patsubst %.c,%,$(wildcard src/*.c))
-PG_CONFIG    = pg_config
+PG_CONFIG    = /usr/pgsql-9.4/bin/pg_config
 
 EXTNDIR     = $(shell $(PG_CONFIG) --sharedir)
 PG91         = $(shell $(PG_CONFIG) --version | grep -qE " 8\.| 9\.0" && echo no || echo yes)
