--- src/gaiageo/gg_transform.c.old	2019-09-03 10:01:50.670929280 +0100
+++ src/gaiageo/gg_transform.c	2019-09-03 10:02:30.536398827 +0100
@@ -55,6 +55,7 @@
 #endif
 
 #ifndef OMIT_PROJ		/* including PROJ.4 */
+#define ACCEPT_USE_OF_DEPRECATED_PROJ_API_H
 #include <proj_api.h>
 #endif
 
--- src/connection_cache/alloc_cache.c.old	2019-09-03 10:03:58.207630616 +0100
+++ src/connection_cache/alloc_cache.c	2019-09-03 10:04:09.921062403 +0100
@@ -81,6 +81,7 @@
 #endif
 
 #ifndef OMIT_PROJ		/* including PROJ.4 */
+#define ACCEPT_USE_OF_DEPRECATED_PROJ_API_H
 #include <proj_api.h>
 #endif
 
--- src/spatialite/spatialite.c.old	2019-09-03 10:04:48.569487084 +0100
+++ src/spatialite/spatialite.c	2019-09-03 10:05:12.491368904 +0100
@@ -121,6 +121,7 @@
 #endif
 
 #ifndef OMIT_PROJ		/* including PROJ.4 */
+#define ACCEPT_USE_OF_DEPRECATED_PROJ_API_H
 #include <proj_api.h>
 #endif
 
--- test/check_sql_stmt.c.old	2019-09-03 10:05:46.449620693 +0100
+++ test/check_sql_stmt.c	2019-09-03 10:05:56.143978052 +0100
@@ -67,6 +67,7 @@
 #endif
 
 #ifndef OMIT_PROJ		/* only if PROJ is supported */
+#define ACCEPT_USE_OF_DEPRECATED_PROJ_API_H
 #include <proj_api.h>
 #endif
 
--- src/gaiageo/gg_geodesic.c.old	2019-09-03 10:08:27.783280716 +0100
+++ src/gaiageo/gg_geodesic.c	2019-09-03 10:08:41.073741627 +0100
@@ -65,6 +65,7 @@
 #include <spatialite_private.h>
 
 #ifndef OMIT_PROJ
+#define ACCEPT_USE_OF_DEPRECATED_PROJ_API_H
 #include <proj_api.h>
 #if defined(PJ_VERSION) && PJ_VERSION >= 490
 /* Enable new proj.4's geodesic distance */
