--- Makefile.old	2024-08-21 12:05:07.592942364 +0300
+++ Makefile	2024-08-21 12:36:21.454636100 +0300
@@ -13,23 +13,23 @@
 
 JDBC_CONFIG = jdbc_config
 
-LIBDIR=/usr/lib64/
+LIBDIR=/usr/lib/jvm/jre-1.8.0-openjdk/lib/amd64/server
 
-SHLIB_LINK += -L$(LIBDIR) -ljvm
+SHLIB_LINK += -L$(LIBDIR) -Wl,-rpath,$(LIBDIR) -ljvm
 
 UNAME = $(shell uname)
 
 TRGTS = JAVAFILES
 
 JAVA_SOURCES = \
-	JDBCUtils.java \
-	JDBCDriverLoader.java \
-	JDBCConnection.java \
-	resultSetInfo.java
+	$(srcdir)/JDBCUtils.java \
+	$(srcdir)/JDBCDriverLoader.java \
+	$(srcdir)/JDBCConnection.java \
+	$(srcdir)/resultSetInfo.java
 
-PG_CPPFLAGS=-D'PKG_LIB_DIR=$(pkglibdir)' -I$(libpq_srcdir)
+PG_CPPFLAGS=-D'PKG_LIB_DIR=$(shell pg_config --sharedir)/extension' -I$(libpq_srcdir)
 
-JFLAGS = -d $(pkglibdir)
+JFLAGS = #-d $(pkglibdir)
 
 all:$(TRGTS)
 
