--- repmgr.conf.sample.old	2015-05-01 14:47:50.244400552 +0300
+++ repmgr.conf.sample	2015-05-01 14:50:05.917954944 +0300
@@ -17,11 +17,11 @@
 # (Note: we recommend to avoid naming nodes after their initial
 #  replication funcion, as this will cause confusion when e.g.
 #  "standby2" is promoted to master)
-node=2
-node_name=node2
+node=1
+node_name=node1
 
 # Database connection information
-conninfo='host=192.168.204.104 dbname=repmgr_db user=repmgr_usr'
+conninfo='host=localhost dbname=repmgr_db user=repmgr_usr'
 
 # Optional configuration items
 # ============================
@@ -55,7 +55,7 @@
 
 # stderr can be redirected to an arbitrary file:
 #
-# logfile='/var/log/repmgr.log'
+logfile='/var/log/repmgr/repmgr-9.3.log'
 
 # event notifications can be passed to an arbitrary external program
 # together with the following parameters:
@@ -82,7 +82,7 @@
 
 # path to PostgreSQL binary directory (location of pg_ctl, pg_basebackup etc.)
 # (if not provided, defaults to system $PATH)
-# pg_bindir=/usr/bin/
+pg_bindir=/usr/pgsql-9.3/bin
 
 # external command options
 
@@ -120,9 +120,9 @@
 # Autofailover options
 failover=automatic  # one of 'automatic', 'manual'
 priority=100        # a value of zero or less prevents the node being promoted to master
-promote_command='repmgr standby promote -f /path/to/repmgr.conf'
-follow_command='repmgr standby follow -f /path/to/repmgr.conf -W'
-
+promote_command='/usr/pgsql-9.3/bin/repmgr standby promote -f /etc/repmgr/9.3/repmgr.conf'
+follow_command='/usr/pgsql-9.3/bin/repmgr standby follow -f /etc/repmgr/9.3/repmgr.conf'
+ 
 # monitoring interval; default is 2s
 #
 # monitor_interval_secs=2
