--- Makefile.old	2017-11-13 09:09:50.126882266 +0000
+++ Makefile	2017-11-13 09:10:19.902063365 +0000
@@ -12,7 +12,7 @@
 endif
 
 ifdef USE_PGXS
-PG_CONFIG = pg_config
+PG_CONFIG = /usr/pgsql-11/bin/pg_config
 PGXS := $(shell $(PG_CONFIG) --pgxs)
 include $(PGXS)
 else
--- lib/Makefile.old	2017-11-13 09:09:54.724910233 +0000
+++ lib/Makefile	2017-11-13 09:10:31.301132697 +0000
@@ -51,7 +51,7 @@
 endif
 
 ifdef USE_PGXS
-PG_CONFIG = pg_config
+PG_CONFIG = /usr/pgsql-11/bin/pg_config
 PGXS := $(shell $(PG_CONFIG) --pgxs)
 include $(PGXS)
 else
--- bin/Makefile.old	2017-11-13 09:09:52.210894942 +0000
+++ bin/Makefile	2017-11-13 09:10:27.410109031 +0000
@@ -21,7 +21,7 @@
 endif
 
 ifdef USE_PGXS
-PG_CONFIG = pg_config
+PG_CONFIG = /usr/pgsql-11/bin/pg_config
 PGXS := $(shell $(PG_CONFIG) --pgxs)
 include $(PGXS)
 else
--- util/Makefile.old	2017-11-13 09:09:58.755934750 +0000
+++ util/Makefile	2017-11-13 09:10:24.343090377 +0000
@@ -20,7 +20,7 @@
 endif
 
 ifdef USE_PGXS
-PG_CONFIG = pg_config
+PG_CONFIG = /usr/pgsql-11/bin/pg_config
 PGXS := $(shell $(PG_CONFIG) --pgxs)
 include $(PGXS)
 else
