--- autogen.sh.old	2021-09-14 12:37:39.388417488 +0100
+++ autogen.sh	2021-09-14 12:38:37.904446721 +0100
@@ -124,12 +124,12 @@
 		exit 0
 	fi
 elif [ "--with-master" == $1 ]; then
-	checkout_mongo_driver &&
+#	checkout_mongo_driver &&
 	checkout_json_lib &&
-	install_mongoc_driver &&
+#	install_mongoc_driver &&
 	install_json_lib &&
 	create_config &&
-	export PKG_CONFIG_PATH=mongo-c-driver/src/libmongoc/src:mongo-c-driver/src/libbson/src &&
+	export PKG_CONFIG_PATH=/usr/pgdg-libmongoc/lib64/pkgconfig
 	cp Makefile.meta Makefile
 
 	ret=$?
