--- src/backend/distributed/utils/statistics_collection.c.old	2019-06-04 21:17:10.770079429 +0100
+++ src/backend/distributed/utils/statistics_collection.c	2019-06-04 21:17:39.381252137 +0100
@@ -15,7 +15,7 @@
 #include "utils/uuid.h"
 #include "utils/backend_random.h"
 
-bool EnableStatisticsCollection = true; /* send basic usage statistics to Citus */
+bool EnableStatisticsCollection = false; /* send basic usage statistics to Citus */
 
 PG_FUNCTION_INFO_V1(citus_server_id);
 
--- src/backend/distributed/shared_library_init.c.old	2019-06-04 21:22:11.577895214 +0100
+++ src/backend/distributed/shared_library_init.c	2019-06-04 21:22:19.778944718 +0100
@@ -1046,7 +1046,7 @@
 					 "whether these reports are sent."),
 		&EnableStatisticsCollection,
 #ifdef HAVE_LIBCURL
-		true,
+		false,
 #else
 		false,
 #endif
