--- pgadmin/db/pgConn.cpp.old	2016-11-11 09:19:11.623252356 +0300
+++ pgadmin/db/pgConn.cpp	2016-11-11 09:23:23.316047020 +0300
@@ -407,21 +407,20 @@
 {
 	// to retrieve edb flag
 	BackendMinimumVersion(0, 0);
-	return isEdb;
+	return BackendMinimumVersion(0, 0) && isEdb;
 }
 
 bool pgConn::GetIsGreenplum()
 {
 	// to retrieve Greenplum flag
 	BackendMinimumVersion(0, 0);
-	return isGreenplum;
-}
+	return BackendMinimumVersion(0, 0) && isGreenplum;}
 
 bool pgConn::GetIsHawq()
 {
 	// to retrieve Greenplum HAWQ flag
 	BackendMinimumVersion(0, 0);
-	return isHawq;
+	return BackendMinimumVersion(0, 0) && isHawq;
 }
 
 wxString pgConn::SystemNamespaceRestriction(const wxString &nsp)
@@ -475,6 +474,9 @@
 
 bool pgConn::BackendMinimumVersion(int major, int minor)
 {
+	if (GetStatus() != PGCONN_OK)
+		return false;
+
 	if (!majorVersion)
 	{
 		wxString version = GetVersionString();
@@ -503,8 +505,8 @@
 // releases are not coordinated with PostgreSQL minor releases.
 bool pgConn::BackendMinimumVersion(int major, int minor, int patch)
 {
-	if (!majorVersion)
-		BackendMinimumVersion(0, 0);
+	if (!BackendMinimumVersion(major, minor))
+	  return false; // false if non connected
 
 	return majorVersion > major || (majorVersion == major && minorVersion > minor) || (majorVersion == major && minorVersion == minor && patchVersion >= patch);
 }
@@ -648,6 +650,11 @@
 	}
 }
 
+int pgConn::GetPort() const
+{
+	return conn ? atoi(PQport(conn)) : 0;
+}
+
 wxString pgConn::GetName() const
 {
 	wxString str;
@@ -1003,13 +1010,13 @@
 }
 
 
-int pgConn::GetStatus() const
+int pgConn::GetStatus(const pgConn *p)
 {
-	if (!this)
+	if (!p)
 		return PGCONN_BAD;
 
-	if (conn)
-		((pgConn *)this)->connStatus = PQstatus(conn);
+	if (p->conn)    // FIXME: casting away const !
+		((pgConn *)p)->connStatus = PQstatus(p->conn);
 
 	return connStatus;
 }
--- pgadmin/include/db/pgConn.h.old	2016-11-11 09:38:01.326354763 +0300
+++ pgadmin/include/db/pgConn.h	2016-11-11 09:39:01.298829343 +0300
@@ -194,10 +194,7 @@
 	{
 		return utfConnectString;
 	}
-	int GetPort() const
-	{
-		return conn ? atoi(PQport(conn)) : 0;
-	};
+	int GetPort() const;
 	wxString GetTTY() const
 	{
 		return conn ? wxString(PQtty(conn), *conv) : wxT("");
@@ -215,7 +212,11 @@
 	{
 		return conn ? PQbackendPID(conn) : 0;
 	}
-	int GetStatus() const;
+	static int GetStatus(const pgConn *);
+	int GetStatus() const
+	{
+		return GetStatus(this);
+	}
 	int GetLastResultStatus() const
 	{
 		return lastResultStatus;
