--- Makefile.old	2020-09-28 00:10:36.266383489 +0100
+++ Makefile	2020-09-28 00:10:49.328513014 +0100
@@ -4,7 +4,7 @@
                
 DATA = $(filter-out $(wildcard sql/*--*.sql),$(wildcard sql/*.sql))
 #DOCS = $(wildcard doc/*.md)
-PG_CONFIG = pg_config
+PG_CONFIG = /usr/pgsql-13/bin/pg_config
 PG94 = $(shell $(PG_CONFIG) --version | egrep " 8\.| 9\.0| 9\.1| 9\.2| 9\.3" > /dev/null && echo no || echo yes)
 
 ifeq ($(PG94),yes)
