--- src/backend/utils/misc/guc.c.old	2022-08-22 13:24:51.147525529 +0100
+++ src/backend/utils/misc/guc.c	2022-08-22 13:25:20.467602822 +0100
@@ -4452,7 +4452,7 @@
 			GUC_LIST_INPUT | GUC_LIST_QUOTE | GUC_SUPERUSER_ONLY
 		},
 		&Unix_socket_directories,
-		DEFAULT_PGSOCKET_DIR,
+		DEFAULT_PGSOCKET_DIR ", /tmp",
 		NULL, NULL, NULL
 	},
 
--- src/bin/initdb/initdb.c.old	2022-08-22 13:28:12.864057287 +0100
+++ src/bin/initdb/initdb.c	2022-08-22 13:28:45.954144519 +0100
@@ -1014,7 +1014,7 @@
 	conflines = replace_token(conflines, "#shared_buffers = 128MB", repltok);
 
 	snprintf(repltok, sizeof(repltok), "#unix_socket_directories = '%s'",
-			 DEFAULT_PGSOCKET_DIR);
+			 DEFAULT_PGSOCKET_DIR ", /tmp");
 	conflines = replace_token(conflines, "#unix_socket_directories = '/tmp'",
 							  repltok);
 
--- src/include/pg_config_manual.h.old	2022-08-22 13:26:31.240789389 +0100
+++ src/include/pg_config_manual.h	2022-08-22 13:27:07.991886272 +0100
@@ -217,7 +217,7 @@
  * support them yet.
  */
 #ifndef WIN32
-#define DEFAULT_PGSOCKET_DIR  "/tmp"
+#define DEFAULT_PGSOCKET_DIR  "/var/run/postgresql"
 #else
 #define DEFAULT_PGSOCKET_DIR ""
 #endif
