--- src/backend/utils/misc/guc_tables.c.old	2022-09-20 22:34:29.420001301 +0100
+++ src/backend/utils/misc/guc_tables.c	2022-09-20 22:34:53.154027380 +0100
@@ -4203,7 +4203,7 @@
 			GUC_LIST_INPUT | GUC_LIST_QUOTE | GUC_SUPERUSER_ONLY
 		},
 		&Unix_socket_directories,
-		DEFAULT_PGSOCKET_DIR,
+		DEFAULT_PGSOCKET_DIR ", /tmp",
 		NULL, NULL, NULL
 	},
 
--- src/bin/initdb/initdb.c.old	2022-09-20 22:39:09.920309535 +0100
+++ src/bin/initdb/initdb.c	2022-09-20 22:39:24.939326045 +0100
@@ -1015,7 +1015,7 @@
 	conflines = replace_token(conflines, "#shared_buffers = 128MB", repltok);
 
 	snprintf(repltok, sizeof(repltok), "#unix_socket_directories = '%s'",
-			 DEFAULT_PGSOCKET_DIR);
+			 DEFAULT_PGSOCKET_DIR ", /tmp");
 	conflines = replace_token(conflines, "#unix_socket_directories = '/tmp'",
 							  repltok);
 
--- src/include/pg_config_manual.h.old	2022-09-20 22:41:25.718479725 +0100
+++ src/include/pg_config_manual.h	2022-09-20 22:41:53.896517110 +0100
@@ -206,7 +206,7 @@
  * support them yet.
  */
 #ifndef WIN32
-#define DEFAULT_PGSOCKET_DIR  "/tmp"
+#define DEFAULT_PGSOCKET_DIR  "/var/run/postgresql"
 #else
 #define DEFAULT_PGSOCKET_DIR ""
 #endif
