--- Makefile.old	2016-10-03 18:28:08.278690023 +0300
+++ Makefile	2016-10-03 18:28:16.993619422 +0300
@@ -8,7 +8,7 @@
 MANDIR = /usr/share/man/man1
 
 # PostgreSQL stuff
-PGCONFIG ?= pg_config
+PGCONFIG ?= /usr/pgsql-9.5/bin/pg_config
 PGLIBDIR = $(shell $(PGCONFIG) --libdir)
 PGINCLUDEDIR = $(shell $(PGCONFIG) --includedir)
 PGLIBS = -lpq
@@ -74,4 +74,4 @@
 
 uninstall:
 	rm -f $(DESTDIR)$(PREFIX)/bin/$(PROGRAM_NAME)
-	rm -f $(MANDIR)/$(PROGRAM_NAME).1.gz
\ No newline at end of file
+	rm -f $(MANDIR)/$(PROGRAM_NAME).1.gz
