--- Makefile.old	2018-12-06 03:59:14.539705987 +0000
+++ Makefile	2018-12-06 03:59:28.105781409 +0000
@@ -19,6 +19,6 @@
 EXTENSION = hdfs_fdw
 DATA = hdfs_fdw--2.0.2.sql hdfs_fdw--2.0.3.sql hdfs_fdw--2.0.1--2.0.2.sql hdfs_fdw--2.0.2--2.0.3.sql hdfs_fdw--2.0.1.sql hdfs_fdw--2.0--2.0.1.sql hdfs_fdw--1.0--2.0.sql hdfs_fdw--1.0.sql
 
-PG_CONFIG = pg_config
+PG_CONFIG = /usr/pgsql-12/bin/pg_config
 PGXS := $(shell $(PG_CONFIG) --pgxs)
 include $(PGXS)
--- libhive/Makefile.old	2018-12-06 03:59:32.538806055 +0000
+++ libhive/Makefile	2018-12-06 03:59:46.246882265 +0000
@@ -1,6 +1,6 @@
 .PHONY : clean
 
-PG_INC_PATH=$(shell pg_config --includedir-server)
+PG_INC_PATH=$(shell /usr/pgsql-12/bin/pg_config --includedir-server)
 CPPFLAGS= -Wno-unused-variable -fPIC -Wall -g -I$(PG_INC_PATH) -I$(JDK_INCLUDE) -I$(JDK_INCLUDE)/linux/ -Ijdbc
 LDFLAGS:= $(LDFLAGS) -shared
 
