--- CMakeLists.txt.old	2021-02-15 11:17:15.326906177 +0000
+++ CMakeLists.txt	2021-02-15 11:17:29.173906252 +0000
@@ -169,7 +169,7 @@
     DOC "The path to a PostgreSQL installation")
 endif()
 
-find_program(PG_CONFIG pg_config
+find_program(PG_CONFIG /usr/pgsql-11/bin/pg_config
   HINTS ${PG_PATH}
   PATH_SUFFIXES bin
   DOC "The path to the pg_config of the PostgreSQL version to compile against")
