--- CMakeLists.txt.old	2020-04-02 10:42:37.178368447 +0100
+++ CMakeLists.txt	2020-04-02 10:43:04.289545775 +0100
@@ -166,7 +166,7 @@
     DOC "The path to a PostgreSQL installation")
 endif()
 
-find_program(PG_CONFIG pg_config
+find_program(PG_CONFIG /usr/pgsql-12/bin/pg_config
   HINTS ${PG_PATH}
   PATH_SUFFIXES bin
   DOC "The path to the pg_config of the PostgreSQL version to compile against")
