--- install/defaultParameter.php.old	2017-03-06 00:21:31.911257219 +0300
+++ install/defaultParameter.php	2017-03-06 00:22:16.217891910 +0300
@@ -23,17 +23,17 @@
  * @version    CVS: $Id$
  */
 
-define("_PGPOOL2_CONFIG_FILE", "/usr/local/etc/pgpool.conf");
-define("_PGPOOL2_PASSWORD_FILE", "/usr/local/etc/pcp.conf");
-define("_PGPOOL2_COMMAND", "/usr/local/bin/pgpool");
+define("_PGPOOL2_CONFIG_FILE", "/etc/pgpool-II-10/pgpool.conf");
+define("_PGPOOL2_PASSWORD_FILE", "/etc/pgpool-II-10/pcp.conf");
+define("_PGPOOL2_COMMAND", "/usr/pgpool-10/bin/pgpool");
 define("_PGPOOL2_CMD_OPTION_C", "0");
 define("_PGPOOL2_CMD_OPTION_LARGE_D", "0");
 define("_PGPOOL2_CMD_OPTION_D", "0");
 define("_PGPOOL2_CMD_OPTION_M", "s");
 define("_PGPOOL2_CMD_OPTION_N", "0");
 define("_PGPOOL2_CMD_OPTION_LARGE_C", "0");
-define("_PGPOOL2_LOG_FILE", "/tmp/pgpool.log");
-define("_PGPOOL2_PCP_DIR", "/usr/local/bin");
+define("_PGPOOL2_LOG_FILE", "/var/log/pgpool-II-10/");
+define("_PGPOOL2_PCP_DIR", "/etc/pgpool-II-10/bin");
 define("_PGPOOL2_PCP_HOSTNAME", "localhost");
 define("_PGPOOL2_PCP_TIMEOUT", "10");
 define("_PGPOOL2_STATUS_REFRESH_TIME", "0");
--- conf/pgmgt.conf.php.old	2017-03-06 00:21:43.250163729 +0300
+++ conf/pgmgt.conf.php	2017-03-06 00:22:29.525782187 +0300
@@ -1,17 +1,17 @@
 <?php
-define('_PGPOOL2_LANG', 'ja');
-define('_PGPOOL2_VERSION', '3.5');
-define('_PGPOOL2_CONFIG_FILE', '/usr/local/etc/pgpool.conf');
-define('_PGPOOL2_PASSWORD_FILE', '/usr/local/etc/pcp.conf');
-define('_PGPOOL2_COMMAND', '/usr/local/bin/pgpool');
+define('_PGPOOL2_LANG', 'en');
+define('_PGPOOL2_VERSION', '3.6');
+define('_PGPOOL2_CONFIG_FILE', '/etc/pgpool-II-10/pgpool.conf');
+define('_PGPOOL2_PASSWORD_FILE', '/etc/pgpool-II-10/pcp.conf');
+define('_PGPOOL2_COMMAND', '/usr/pgpool-10/bin/pgpool');
 define('_PGPOOL2_CMD_OPTION_C', '0');
 define('_PGPOOL2_CMD_OPTION_LARGE_D', '0');
 define('_PGPOOL2_CMD_OPTION_D', '0');
 define('_PGPOOL2_CMD_OPTION_M', 's');
 define('_PGPOOL2_CMD_OPTION_N', '0');
 define('_PGPOOL2_CMD_OPTION_LARGE_C', '0');
-define('_PGPOOL2_LOG_FILE', '/tmp/pgpool.log');
-define('_PGPOOL2_PCP_DIR', '/usr/local/bin');
+define('_PGPOOL2_LOG_FILE', '/var/log/pgpool-II-10/');
+define('_PGPOOL2_PCP_DIR', '/usr/pgpool-10/bin');
 define('_PGPOOL2_PCP_HOSTNAME', 'localhost');
 define('_PGPOOL2_STATUS_REFRESH_TIME', '0');
 ?>
--- definePgpoolConfParam.php.old	2017-03-06 00:21:51.469095964 +0300
+++ definePgpoolConfParam.php	2017-03-06 00:22:16.218891902 +0300
@@ -333,7 +333,7 @@
 
 $key = 'pid_file_name';
 $pgpoolConfigParam[$key]['type'] = 'C';
-$pgpoolConfigParam[$key]['default'] = '/var/run/pgpool/pgpool.pid';
+$pgpoolConfigParam[$key]['default'] = '/var/run/pgpool-II-10/pgpool.pid';
 $pgpoolConfigParam[$key]['regexp'] = $dirreg;
 $pgpoolConfigParam[$key]['restart'] = TRUE;
 
