--- web/pgadmin/feature_tests/xss_checks_panels_and_query_tool_test.py.old	2019-01-25 06:20:48.923171236 +0000
+++ web/pgadmin/feature_tests/xss_checks_panels_and_query_tool_test.py	2019-01-25 06:21:01.397569305 +0000
@@ -7,6 +7,7 @@
 #
 ##########################################################################
 
+from __future__ import print_function
 from regression.python_test_utils import test_utils
 from regression.feature_utils.base_feature_test import BaseFeatureTest
 from selenium.webdriver import ActionChains
