--- docs/en_US/conf.py.old	2019-07-23 22:38:28.614806517 +0100
+++ docs/en_US/conf.py	2019-07-23 22:38:45.267908976 +0100
@@ -107,7 +107,7 @@
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = 'pgadmin4'
+html_theme = 'default'
 
 # Add any paths that contain custom themes here, relative to this directory.
 html_theme_path = ['./theme']   # make sphinx search for themes in current dir
