--- src/sample/pgpool.conf.sample-stream.old	2021-09-20 23:11:35.862450313 +0100
+++ src/sample/pgpool.conf.sample-stream	2021-09-20 23:12:22.247599083 +0100
@@ -80,7 +80,7 @@
                                    # Port number for backend 0
 backend_weight0 = 1
                                    # Weight for backend 0 (only in load balancing mode)
-backend_data_directory0 = '/data'
+backend_data_directory0 = '/var/lib/pgsql/14/data'
                                    # Data directory for backend 0
 backend_flag0 = 'ALLOW_TO_FAILOVER'
                                    # Controls various backend behavior
@@ -292,13 +292,13 @@
 # FILE LOCATIONS
 #------------------------------------------------------------------------------
 
-pid_file_name = '/var/run/pgpool/pgpool.pid'
+pid_file_name = '/var/run/pgpool-II-42_14/pgpool.pid'
                                    # PID file name
                                    # Can be specified as relative to the"
                                    # location of pgpool.conf file or
                                    # as an absolute path
                                    # (change requires restart)
-logdir = '/tmp'
+logdir = '/var/log/pgpool-II-42_14'
                                    # Directory of pgPool status file
                                    # (change requires restart)
 
