--- Makefile.old	2022-06-11 20:52:42.086532078 +0100
+++ Makefile	2022-06-11 20:53:02.980592642 +0100
@@ -30,7 +30,7 @@
 python_code: setup.py
 	cp $(srcdir)/setup.py ./setup--$(EXTVERSION).py
 	sed -i -e "s/__VERSION__/$(EXTVERSION)/g" ./setup--$(EXTVERSION).py
-	$(PYTHON) ./setup--$(EXTVERSION).py install
+	$(PYTHON) ./setup--$(EXTVERSION).py install --root=$(DESTDIR)
 	rm ./setup--$(EXTVERSION).py
 
 release-zip: all
