From d57b9989862329bfc4ff0c70ea52c7b1200ff90d Mon Sep 17 00:00:00 2001
From: Denis Lussier <denis@lussier.io>
Date: Sat, 8 Jun 2024 07:21:47 +0000
Subject: [PATCH] fix compilation on pg17

---
 src/multicorn.c | 6 ++++++
 src/query.c     | 3 +++
 2 files changed, 9 insertions(+)

diff --git a/src/multicorn.c b/src/multicorn.c
index 9851f6d..4b1a568 100644
--- a/src/multicorn.c
+++ b/src/multicorn.c
@@ -368,6 +368,9 @@ multicornGetForeignPaths(PlannerInfo *root,
 			NIL,		/* no pathkeys */
 			NULL,
 			NULL,
+#if PG_VERSION_NUM >= 170000
+			NULL,
+#endif
 			NULL));
 
 	/* Handle sort pushdown */
@@ -402,6 +405,9 @@ multicornGetForeignPaths(PlannerInfo *root,
 					path->path.startup_cost, path->path.total_cost,
 					apply_pathkeys, NULL,
 					NULL,
+#if PG_VERSION_NUM >= 170000
+					NULL,
+#endif
 					(void *) deparsed_pathkeys);
 
 			newpath->path.param_info = path->path.param_info;
diff --git a/src/query.c b/src/query.c
index b6e7a8a..af787b8 100644
--- a/src/query.c
+++ b/src/query.c
@@ -721,6 +721,9 @@ findPaths(PlannerInfo *root, RelOptInfo *baserel, List *possiblePaths,
 					NIL, /* no pathkeys */
 					NULL,
 					NULL,
+#if PG_VERSION_NUM >= 170000
+					NULL,
+#endif
 					NULL);
 
 				foreignPath->path.param_info = ppi;
