--- Makefile.old	2024-06-25 16:26:28.228092150 +0300
+++ Makefile	2024-06-25 16:26:50.966455961 +0300
@@ -29,21 +29,6 @@
 
 python_code: setup.py pyproject.toml
 	$(eval python_major_version := $(shell echo ${python_version} | cut -d '.' -f 1))
-	$(eval PIP ?= $(shell ([ -x "$$(command -v pip${python_version})" ] && echo pip${python_version}) || ([ -x "$$(command -v pip${python_major_version})" ] && echo pip${python_major_version}) || echo pip))
-	#
-	# Strictly speaking, --break-system-packages arrived in 23.0.1, but that's
-	# too hard to check for.
-	#
-	$(eval PIP_VERSION := $(shell $(PIP) --version 2>&1 | cut -d ' ' -f 2 | cut -d '.' -f 1))
-	$(eval PIP_FLAGS := $(shell [ $(PIP_VERSION) -ge 23 ] && echo "--break-system-packages"))
-	#
-	# Workaround https://github.com/pgsql-io/multicorn2/issues/34, and then
-	# re-evaluate PIP_VERSION/PIP_FLAGS.
-	#
-	$(PIP) install $(PIP_FLAGS) --upgrade 'pip>=23'
-	$(eval PIP_VERSION := $(shell $(PIP) --version 2>&1 | cut -d ' ' -f 2 | cut -d '.' -f 1))
-	$(eval PIP_FLAGS := $(shell [ $(PIP_VERSION) -ge 23 ] && echo "--break-system-packages"))
-	$(PIP) install $(PIP_FLAGS) .
 
 release-zip: all
 	git archive --format zip --prefix=multicorn-$(EXTVERSION)/ --output ./multicorn-$(EXTVERSION).zip HEAD
