--- tools/splitconfig.old	2019-09-06 22:48:56.137042124 +0100
+++ tools/splitconfig	2019-09-06 22:49:19.758701203 +0100
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/python3
 
 """Extract configuration items into various configuration headers.
 
--- tools/template2mak.py.old	2019-09-06 22:49:00.462162804 +0100
+++ tools/template2mak.py	2019-09-06 22:49:14.922566269 +0100
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/python3
 """Minimal macro processor.  Used for generating VC++ makefiles.
 
 The available template commands are:
