--- postgresql-plruby-0.5.4/extconf.rb.rbconfig	2015-01-17 02:16:39.379409665 +0900
+++ postgresql-plruby-0.5.4/extconf.rb	2015-01-17 02:21:42.793184523 +0900
@@ -52,7 +52,7 @@
 
 
    create function plruby#{suffix}_call_handler() returns #{opaque}
-   as '#{Config::CONFIG["sitearchdir"]}/plruby#{suffix}.#{CONFIG["DLEXT"]}'
+   as '#{RbConfig::CONFIG["sitearchdir"]}/plruby#{suffix}.#{RbConfig::CONFIG["DLEXT"]}'
    language '#{language}';
 
    create #{trusted} language 'plruby#{suffix}'
@@ -190,7 +190,7 @@
    orig_argv << "--with-cflags='#$CFLAGS -I.. -I ../..'"
    orig_argv << "--with-ldflags='#$LDFLAGS'"
    orig_argv << "--with-cppflags='#$CPPFLAGS'"
-   cmd = "#{CONFIG['RUBY_INSTALL_NAME']} extconf.rb #{orig_argv.join(' ')}"
+   cmd = "#{RbConfig::CONFIG['RUBY_INSTALL_NAME']} extconf.rb #{orig_argv.join(' ')}"
    system("cd #{key}; #{cmd}")
 end
 
@@ -198,13 +198,13 @@
 
 begin
    Dir.chdir("src")
-   if CONFIG["ENABLE_SHARED"] == "no"
-      libs = if CONFIG.key?("LIBRUBYARG_STATIC")
-                Config::expand(CONFIG["LIBRUBYARG_STATIC"].dup).sub(/^-l/, '')
+   if RbConfig::CONFIG["ENABLE_SHARED"] == "no"
+      libs = if RbConfig::CONFIG.key?("LIBRUBYARG_STATIC")
+                RbConfig::expand(RbConfig::CONFIG["LIBRUBYARG_STATIC"].dup).sub(/^-l/, '')
              else
-                Config::expand(CONFIG["LIBRUBYARG"].dup).sub(/lib([^.]*).*/, '\\1')
+                RbConfig::expand(RbConfig::CONFIG["LIBRUBYARG"].dup).sub(/lib([^.]*).*/, '\\1')
              end
-      find_library(libs, "ruby_init", Config::expand(CONFIG["archdir"].dup))
+      find_library(libs, "ruby_init", RbConfig::expand(RbConfig::CONFIG["archdir"].dup))
    end
    $objs = ["plruby.o", "plplan.o", "plpl.o", "pltrans.o"] unless $objs
    create_makefile("plruby#{suffix}")
--- postgresql-plruby-0.5.4/src/conversions/basic/extconf.rb.rbconfig	2010-01-02 04:08:20.000000000 +0900
+++ postgresql-plruby-0.5.4/src/conversions/basic/extconf.rb	2015-01-17 02:23:20.917758514 +0900
@@ -1,4 +1,5 @@
 require 'mkmf'
+include RbConfig
 
 if CONFIG["LIBRUBYARG"] == "$(LIBRUBYARG_SHARED)" && 
       !enable_config("plruby-shared")
--- postgresql-plruby-0.5.4/src/conversions/bitstring/extconf.rb.rbconfig	2010-01-02 04:08:20.000000000 +0900
+++ postgresql-plruby-0.5.4/src/conversions/bitstring/extconf.rb	2015-01-17 02:22:31.577469892 +0900
@@ -1,4 +1,5 @@
 require 'mkmf'
+include RbConfig
 
 if CONFIG["LIBRUBYARG"] == "$(LIBRUBYARG_SHARED)" && 
       !enable_config("plruby-shared")
--- postgresql-plruby-0.5.4/src/conversions/datetime/extconf.rb.rbconfig	2010-01-02 04:08:20.000000000 +0900
+++ postgresql-plruby-0.5.4/src/conversions/datetime/extconf.rb	2015-01-17 02:17:36.850745851 +0900
@@ -1,4 +1,5 @@
 require 'mkmf'
+include RbConfig
 
 if CONFIG["LIBRUBYARG"] == "$(LIBRUBYARG_SHARED)" && 
       !enable_config("plruby-shared")
--- postgresql-plruby-0.5.4/src/conversions/geometry/extconf.rb.rbconfig	2010-01-02 04:08:20.000000000 +0900
+++ postgresql-plruby-0.5.4/src/conversions/geometry/extconf.rb	2015-01-17 02:23:24.909781866 +0900
@@ -1,4 +1,5 @@
 require 'mkmf'
+include RbConfig
 
 if CONFIG["LIBRUBYARG"] == "$(LIBRUBYARG_SHARED)" && 
       !enable_config("plruby-shared")
--- postgresql-plruby-0.5.4/src/conversions/network/extconf.rb.rbconfig	2010-01-02 04:08:20.000000000 +0900
+++ postgresql-plruby-0.5.4/src/conversions/network/extconf.rb	2015-01-17 02:22:27.388445388 +0900
@@ -1,4 +1,5 @@
 require 'mkmf'
+include RbConfig
 
 if CONFIG["LIBRUBYARG"] == "$(LIBRUBYARG_SHARED)" && 
       !enable_config("plruby-shared")
--- postgresql-plruby-0.5.4/test/conv_bitstring/b.rb.rbconfig	2010-01-02 04:08:20.000000000 +0900
+++ postgresql-plruby-0.5.4/test/conv_bitstring/b.rb	2015-01-17 02:23:30.873816754 +0900
@@ -1,6 +1,6 @@
 #!/usr/bin/ruby
 require 'rbconfig'
-include Config
+include RbConfig
 pwd = Dir.pwd
 pwd.sub!(%r{[^/]+/[^/]+$}, "")
 
--- postgresql-plruby-0.5.4/test/conv_geometry/b.rb.rbconfig	2010-01-02 04:08:20.000000000 +0900
+++ postgresql-plruby-0.5.4/test/conv_geometry/b.rb	2015-01-17 02:24:13.731067453 +0900
@@ -1,6 +1,6 @@
 #!/usr/bin/ruby
 require 'rbconfig'
-include Config
+include RbConfig
 pwd = Dir.pwd
 pwd.sub!(%r{[^/]+/[^/]+$}, "")
 
--- postgresql-plruby-0.5.4/test/conv_network/b.rb.rbconfig	2010-01-02 04:08:20.000000000 +0900
+++ postgresql-plruby-0.5.4/test/conv_network/b.rb	2015-01-17 02:24:09.054040094 +0900
@@ -1,6 +1,6 @@
 #!/usr/bin/ruby
 require 'rbconfig'
-include Config
+include RbConfig
 pwd = Dir.pwd
 pwd.sub!(%r{[^/]+/[^/]+$}, "")
 
--- postgresql-plruby-0.5.4/test/plp/b.rb.rbconfig	2010-01-02 04:08:20.000000000 +0900
+++ postgresql-plruby-0.5.4/test/plp/b.rb	2015-01-17 02:23:52.894945569 +0900
@@ -1,6 +1,6 @@
 #!/usr/bin/ruby
 require 'rbconfig'
-include Config
+include RbConfig
 pwd = Dir.pwd
 pwd.sub!(%r{[^/]+/[^/]+$}, "")
 
--- postgresql-plruby-0.5.4/test/plt/b.rb.rbconfig	2010-01-02 04:08:20.000000000 +0900
+++ postgresql-plruby-0.5.4/test/plt/b.rb	2015-01-17 02:24:03.033004873 +0900
@@ -1,6 +1,6 @@
 #!/usr/bin/ruby
 require 'rbconfig'
-include Config
+include RbConfig
 pwd = Dir.pwd
 pwd.sub!(%r{[^/]+/[^/]+$}, "")
 
--- postgresql-plruby-0.5.4/test/range/b.rb.rbconfig	2010-01-02 04:08:20.000000000 +0900
+++ postgresql-plruby-0.5.4/test/range/b.rb	2015-01-17 02:24:20.178105166 +0900
@@ -1,6 +1,6 @@
 #!/usr/bin/ruby
 require 'rbconfig'
-include Config
+include RbConfig
 pwd = Dir.pwd
 pwd.sub!(%r{[^/]+/[^/]+$}, "")
 
