--- bin/pgbackrest.old	2016-05-27 17:03:24.407846885 +0300
+++ bin/pgbackrest	2016-05-27 17:03:28.769801904 +0300
@@ -16,7 +16,6 @@
 use File::Basename qw(dirname);
 use Scalar::Util qw(blessed);
 
-use lib dirname($0) . '/../lib';
 use pgBackRest::Archive;
 use pgBackRest::Common::Exception;
 use pgBackRest::Common::Exit;
--- lib/pgBackRest/Common/Exit.pm.old	2016-05-27 17:03:59.499485012 +0300
+++ lib/pgBackRest/Common/Exit.pm	2016-05-27 17:04:05.415424006 +0300
@@ -12,7 +12,6 @@
 use File::Basename qw(dirname);
 use Scalar::Util qw(blessed);
 
-use lib dirname($0) . '/../lib';
 use pgBackRest::Common::Exception;
 use pgBackRest::Common::Lock;
 use pgBackRest::Common::Log;
--- lib/pgBackRest/Common/Lock.pm.old	2016-05-27 17:04:16.166313140 +0300
+++ lib/pgBackRest/Common/Lock.pm	2016-05-27 17:04:21.290260301 +0300
@@ -12,7 +12,6 @@
 use Fcntl qw(:DEFAULT :flock);
 use File::Basename qw(dirname);
 
-use lib dirname($0) . '/../lib';
 use pgBackRest::Common::Exception;
 use pgBackRest::Common::Log;
 use pgBackRest::Common::String;
--- lib/pgBackRest/Common/Log.pm.old	2016-05-27 17:04:29.729173277 +0300
+++ lib/pgBackRest/Common/Log.pm	2016-05-27 17:04:33.645132895 +0300
@@ -15,7 +15,6 @@
 use Scalar::Util qw(blessed reftype);
 use Time::HiRes qw(gettimeofday usleep);
 
-use lib dirname($0) . '/../lib';
 use pgBackRest::Common::Exception;
 use pgBackRest::Common::String;
 
--- lib/pgBackRest/Common/Wait.pm.old	2016-05-27 17:04:43.743028763 +0300
+++ lib/pgBackRest/Common/Wait.pm	2016-05-27 17:04:49.191972572 +0300
@@ -14,7 +14,6 @@
 use POSIX qw(ceil);
 use Time::HiRes qw(gettimeofday usleep);
 
-use lib dirname($0) . '/../lib';
 use pgBackRest::Common::Log;
 
 ####################################################################################################################################
--- lib/pgBackRest/Config/Config.pm.old	2016-05-27 17:06:48.643740758 +0300
+++ lib/pgBackRest/Config/Config.pm	2016-05-27 17:06:56.931655291 +0300
@@ -13,7 +13,6 @@
 use File::Basename qw(dirname basename);
 use Getopt::Long qw(GetOptions);
 
-use lib dirname($0) . '/../lib';
 use pgBackRest::Common::Exception;
 use pgBackRest::Common::Ini;
 use pgBackRest::Common::Log;
--- lib/pgBackRest/Config/ConfigHelp.pm.old	2016-05-27 17:07:09.960520934 +0300
+++ lib/pgBackRest/Config/ConfigHelp.pm	2016-05-27 17:07:16.858449801 +0300
@@ -11,7 +11,6 @@
     our @EXPORT = qw();
 use File::Basename qw(dirname);
 
-use lib dirname($0) . '/../lib';
 use pgBackRest::Common::Exception;
 use pgBackRest::Common::Ini;
 use pgBackRest::Common::Log;
--- lib/pgBackRest/File.pm.old	2016-05-27 17:08:12.660874354 +0300
+++ lib/pgBackRest/File.pm	2016-05-27 17:08:23.776759724 +0300
@@ -17,7 +17,6 @@
 use IO::Handle;
 use Scalar::Util qw(blessed);
 
-use lib dirname($0) . '/../lib';
 use pgBackRest::Common::Exception;
 use pgBackRest::Common::Log;
 use pgBackRest::Common::String;
--- lib/pgBackRest/FileCommon.pm.old	2016-05-27 17:08:32.396670834 +0300
+++ lib/pgBackRest/FileCommon.pm	2016-05-27 17:08:39.775594741 +0300
@@ -16,7 +16,6 @@
 use File::stat;
 use IO::Handle;
 
-use lib dirname($0) . '/../lib';
 use pgBackRest::Common::Exception;
 use pgBackRest::Common::Log;
 
--- lib/pgBackRest/Protocol/CommonMaster.pm.old	2016-05-27 17:09:47.765893609 +0300
+++ lib/pgBackRest/Protocol/CommonMaster.pm	2016-05-27 17:10:03.509731272 +0300
@@ -11,7 +11,6 @@
 use File::Basename qw(dirname);
 use Time::HiRes qw(gettimeofday);
 
-use lib dirname($0) . '/../lib';
 use pgBackRest::Common::Exception;
 use pgBackRest::Common::Ini;
 use pgBackRest::Common::Log;
--- lib/pgBackRest/Protocol/CommonMinion.pm.old	2016-05-27 17:10:23.989523587 +0300
+++ lib/pgBackRest/Protocol/CommonMinion.pm	2016-05-27 17:10:30.769454832 +0300
@@ -11,7 +11,6 @@
 use File::Basename qw(dirname);
 use Scalar::Util qw(blessed);
 
-use lib dirname($0) . '/../lib';
 use pgBackRest::Common::Exception;
 use pgBackRest::Common::Ini;
 use pgBackRest::Common::Log;
--- lib/pgBackRest/Protocol/Common.pm.old	2016-05-27 17:09:28.222095149 +0300
+++ lib/pgBackRest/Protocol/Common.pm	2016-05-27 17:09:33.046045404 +0300
@@ -12,7 +12,6 @@
 use Compress::Raw::Zlib qw(WANT_GZIP Z_OK Z_BUF_ERROR Z_STREAM_END);
 use File::Basename qw(dirname);
 
-use lib dirname($0) . '/../lib';
 use pgBackRest::Common::Exception;
 use pgBackRest::Common::Ini;
 use pgBackRest::Common::Log;
--- lib/pgBackRest/Protocol/IO.pm.old	2016-05-27 17:10:41.661344377 +0300
+++ lib/pgBackRest/Protocol/IO.pm	2016-05-27 17:10:44.831312231 +0300
@@ -16,7 +16,6 @@
 use Symbol 'gensym';
 use Time::HiRes qw(gettimeofday);
 
-use lib dirname($0) . '/../lib';
 use pgBackRest::Common::Exception;
 use pgBackRest::Common::Log;
 use pgBackRest::Common::String;
--- lib/pgBackRest/Protocol/RemoteMaster.pm.old	2016-05-27 17:10:53.981219442 +0300
+++ lib/pgBackRest/Protocol/RemoteMaster.pm	2016-05-27 17:10:58.687171719 +0300
@@ -10,7 +10,6 @@
 
 use File::Basename qw(dirname);
 
-use lib dirname($0) . '/../lib';
 use pgBackRest::Common::Log;
 use pgBackRest::Config::Config;
 use pgBackRest::Protocol::CommonMaster;
--- lib/pgBackRest/Protocol/ThreadGroup.pm.old	2016-05-27 17:11:07.517082175 +0300
+++ lib/pgBackRest/Protocol/ThreadGroup.pm	2016-05-27 17:11:10.888047991 +0300
@@ -12,7 +12,6 @@
     our @EXPORT = qw();
 use File::Basename;
 
-use lib dirname($0) . '/../lib';
 use pgBackRest::Common::Log;
 use pgBackRest::Common::Wait;
 use pgBackRest::Config::Config;
--- lib/pgBackRest/Protocol/RemoteMinion.pm.old	2016-05-27 17:15:33.229387597 +0300
+++ lib/pgBackRest/Protocol/RemoteMinion.pm	2016-05-27 17:15:37.943339793 +0300
@@ -10,7 +10,6 @@
 
 use File::Basename qw(dirname);
 
-use lib dirname($0) . '/../lib';
 use pgBackRest::Common::Exception;
 use pgBackRest::Common::Log;
 use pgBackRest::Archive;
