--- sql/Makefile.old	2014-02-20 03:20:57.286188620 +0200
+++ sql/Makefile	2014-02-20 03:21:03.298144673 +0200
@@ -29,7 +29,7 @@
 top_builddir = ..
 
 ifdef USE_PGXS
-PG_CONFIG = pg_config
+PG_CONFIG = /usr/pgsql-9.0/bin/pg_config
 PGXS = $(shell $(PG_CONFIG) --pgxs)
 include $(PGXS)
 else
--- sql/pgpool-recovery/Makefile.old	2014-02-20 03:21:09.592098664 +0200
+++ sql/pgpool-recovery/Makefile	2014-02-20 03:21:16.036051558 +0200
@@ -14,7 +14,7 @@
 top_builddir = ../..
 
 ifdef USE_PGXS
-PG_CONFIG = pg_config
+PG_CONFIG = /usr/pgsql-9.0/bin/pg_config
 PGXS = $(shell $(PG_CONFIG) --pgxs)
 include $(PGXS)
 else
--- sql/pgpool-regclass/Makefile.old	2014-02-20 03:21:21.777009591 +0200
+++ sql/pgpool-regclass/Makefile	2014-02-20 03:21:27.111970591 +0200
@@ -14,7 +14,7 @@
 top_builddir = ../..
 
 ifdef USE_PGXS
-PG_CONFIG = pg_config
+PG_CONFIG = /usr/pgsql-9.0/bin/pg_config
 PGXS = $(shell $(PG_CONFIG) --pgxs)
 include $(PGXS)
 else
--- pgpool_adm/Makefile.old	2014-02-20 03:20:23.607434799 +0200
+++ pgpool_adm/Makefile	2014-02-20 03:20:43.376290298 +0200
@@ -9,7 +9,7 @@
 SHLIB_LINK = -lpcp
 
 ifdef USE_PGXS
-PG_CONFIG = pg_config
+PG_CONFIG = /usr/pgsql-9.0/bin/pg_config
 PGXS := $(shell $(PG_CONFIG) --pgxs)
 include $(PGXS)
 else
