--- Makefile.old	2016-10-27 10:21:35.216584879 +0300
+++ Makefile	2016-10-27 10:21:48.022467336 +0300
@@ -8,7 +8,7 @@
 REGRESS      = $(patsubst test/sql/%.sql,%,$(TESTS))
 REGRESS_OPTS = --inputdir=test --load-language=plpgsql
 #MODULES      = $(patsubst %.c,%,$(wildcard src/*.c))
-PG_CONFIG    = pg_config
+PG_CONFIG    = /usr/pgsql-10/bin/pg_config
 PG91         = $(shell $(PG_CONFIG) --version | grep -qE " 8\.| 9\.0" && echo no || echo yes)
 
 SQL_BITS     = $(wildcard sql_bits/*.sql)
