--- Makefile.old	2017-06-08 14:30:53.914101123 +0100
+++ Makefile	2017-06-08 14:31:37.962708175 +0100
@@ -39,7 +39,7 @@
 # example: /usr/local/pgsql/bin/pg_config or /usr/lib/postgresql/9.1/bin/pg_config
 #
 
-PG_CONFIG = pg_config
+PG_CONFIG = /usr/pgsql-9.6/bin/pg_config
 PGXS := $(shell $(PG_CONFIG) --pgxs)
 include $(PGXS)
 
--- Makefile.meta.old	2017-06-08 15:08:04.811838956 +0100
+++ Makefile.meta	2017-06-08 15:08:30.692171722 +0100
@@ -18,7 +18,8 @@
                                 $(LIBJSON)/arraylist.o $(LIBJSON)/random_seed.o $(LIBJSON)/debug.o
 
 MONGO_INCLUDE = $(shell pkg-config --cflags libmongoc-1.0)
-PG_CPPFLAGS = --std=c99 $(MONGO_INCLUDE) -I$(LIBJSON) -DMETA_DRIVER
+JSON_INCLUDE = $(shell pkg-config --cflags json-c)
+PG_CPPFLAGS = --std=c99 $(MONGO_INCLUDE) $(JSON_INCLUDE) -I$(LIBJSON) -DMETA_DRIVER
 SHLIB_LINK = $(shell pkg-config --libs libmongoc-1.0)
 
 OBJS = connection.o option.o mongo_wrapper_meta.o mongo_fdw.o mongo_query.o $(LIBJSON_OBJS)
@@ -35,7 +36,7 @@
 # example: /usr/local/pgsql/bin/pg_config or /usr/lib/postgresql/9.1/bin/pg_config
 #
 
-PG_CONFIG = pg_config
+PG_CONFIG = /usr/pgsql-9.6/bin/pg_config
 PGXS := $(shell $(PG_CONFIG) --pgxs)
 include $(PGXS)
 
