--- Makefile.old	2015-04-17 11:49:03.082718026 +0300
+++ Makefile	2015-04-17 11:49:19.516584716 +0300
@@ -9,7 +9,7 @@
 #MODULES      = $(patsubst %.c,%,$(wildcard src/*.c))
 MODULE_big   = $(EXTENSION)
 OBJS 				 = src/ldap_fdw.o
-PG_CONFIG    = `which pg_config`
+PG_CONFIG    = `/usr/pgsql-9.6/bin/pg_config`
 SHLIB_LINK	 += -lldap
 
 PG91         = $(shell $(PG_CONFIG) --version | grep -qE " 8\.| 9\.0" && echo no || echo yes)
