--- Makefile.old	2015-01-19 15:56:05.997893917 +0200
+++ Makefile	2015-01-19 15:56:18.456768020 +0200
@@ -23,7 +23,7 @@
 # You may have to customize this to run the test suite
 # REGRESS_OPTS=--user postgres
 
-PG_CONFIG=pg_config
+PG_CONFIG=/usr/pgsql-11/bin/pg_config
 
 SHLIB_LINK=-lgmp -lm
 
@@ -40,7 +40,7 @@
 SRCFILES = $(SRC_C) $(SRC_H)
 OBJS = $(patsubst %.c,%.o,$(SRC_C))
 TESTFILES = $(wildcard test/sql/*.sql) $(wildcard test/expected/*.out)
-DOCS = $(wildcard docs/*.rst) docs/conf.py docs/Makefile docs/_static/pgmp.css
+DOCS = $(wildcard docs/*.rst)
 
 PKGFILES = AUTHORS COPYING README.rst Makefile \
 	META.json pgmp.control \
