--- install/defaultParameter.php.old	2014-12-14 12:51:25.523701332 +0200
+++ install/defaultParameter.php	2014-12-14 12:52:27.377043709 +0200
@@ -23,9 +23,9 @@
  * @version    CVS: $Id$
  */
 
-define("_PGPOOL2_CONFIG_FILE", "/usr/local/etc/pgpool.conf");
-define("_PGPOOL2_PASSWORD_FILE", "/usr/local/etc/pcp.conf");
-define("_PGPOOL2_COMMAND", "/usr/local/bin/pgpool");
+define("_PGPOOL2_CONFIG_FILE", "/etc/pgpool-II-94/pgpool.conf");
+define("_PGPOOL2_PASSWORD_FILE", "/etc/pgpool-II-94/pcp.conf");
+define("_PGPOOL2_COMMAND", "/usr/pgpool-9.4/bin/pgpool");
 define("_PGPOOL2_CMD_OPTION_C", "0");
 define("_PGPOOL2_CMD_OPTION_LARGE_D", "0");
 define("_PGPOOL2_CMD_OPTION_D", "0");
@@ -33,7 +33,7 @@
 define("_PGPOOL2_CMD_OPTION_N", "0");
 define("_PGPOOL2_CMD_OPTION_LARGE_C", "0");
 define("_PGPOOL2_LOG_FILE", "/tmp/pgpool.log");
-define("_PGPOOL2_PCP_DIR", "/usr/local/bin");
+define("_PGPOOL2_PCP_DIR", "/usr/pgpool-9.4/bin");
 define("_PGPOOL2_PCP_HOSTNAME", "localhost");
 define("_PGPOOL2_PCP_TIMEOUT", "10");
 define("_PGPOOL2_STATUS_REFRESH_TIME", "0");
--- conf/pgmgt.conf.php.old	2014-12-14 12:59:02.202831626 +0200
+++ conf/pgmgt.conf.php	2014-12-14 12:59:57.326241491 +0200
@@ -1,9 +1,9 @@
 <?php
 define('_PGPOOL2_LANG', 'ja');
 define('_PGPOOL2_VERSION', '3.2');
-define('_PGPOOL2_CONFIG_FILE', '/usr/local/etc/pgpool.conf');
-define('_PGPOOL2_PASSWORD_FILE', '/usr/local/etc/pcp.conf');
-define('_PGPOOL2_COMMAND', '/usr/local/bin/pgpool');
+define('_PGPOOL2_CONFIG_FILE', '/etc/pgpool-II-94/pgpool.conf');
+define('_PGPOOL2_PASSWORD_FILE', '/etc/pgpool-II-94/pcp.conf');
+define('_PGPOOL2_COMMAND', '/usr/pgpool-9.4/bin/pgpool');
 define('_PGPOOL2_CMD_OPTION_C', '0');
 define('_PGPOOL2_CMD_OPTION_LARGE_D', '0');
 define('_PGPOOL2_CMD_OPTION_D', '0');
@@ -11,7 +11,7 @@
 define('_PGPOOL2_CMD_OPTION_N', '0');
 define('_PGPOOL2_CMD_OPTION_LARGE_C', '0');
 define('_PGPOOL2_LOG_FILE', '/tmp/pgpool.log');
-define('_PGPOOL2_PCP_DIR', '/usr/local/bin');
+define('_PGPOOL2_PCP_DIR', '/usr/pgpool-9.4/bin');
 define('_PGPOOL2_PCP_HOSTNAME', 'localhost');
 define('_PGPOOL2_STATUS_REFRESH_TIME', '0');
 ?>
