--- Makefile.old	2017-05-28 11:13:31.600663026 +0300
+++ Makefile	2017-05-28 11:13:44.275568943 +0300
@@ -14,7 +14,7 @@
 #-----------------------------------------------------------------------------#
 PLV8_VERSION = 2.0.3
 
-PG_CONFIG = pg_config
+PG_CONFIG = /usr/pgsql-9.3/bin/pg_config
 PGXS := $(shell $(PG_CONFIG) --pgxs)
 
 PG_VERSION_NUM := $(shell cat `$(PG_CONFIG) --includedir-server`/pg_config*.h \
