--- Makefile.old	2017-10-22 13:30:36.981004237 +0100
+++ Makefile	2017-10-22 13:30:43.040955169 +0100
@@ -14,7 +14,7 @@
 #-----------------------------------------------------------------------------#
 PLV8_VERSION = 2.1.0
 
-PG_CONFIG = pg_config
+PG_CONFIG = /usr/pgsql-9.3/bin/pg_config
 PGXS := $(shell $(PG_CONFIG) --pgxs)
 
 PG_VERSION_NUM := $(shell cat `$(PG_CONFIG) --includedir-server`/pg_config*.h \
