--- Makefile.old	2014-02-15 10:55:18.833760610 +0200
+++ Makefile	2014-02-15 10:55:27.611693129 +0200
@@ -5,7 +5,7 @@
 DATA = $(filter-out $(wildcard updates/*--*.sql),$(wildcard sql/*.sql))
 DOCS = $(wildcard doc/*.md)
 SCRIPTS = bin/dump_partition.py bin/partition_data.py bin/reapply_indexes.py bin/undo_partition.py
-PG_CONFIG = pg_config
+PG_CONFIG = /usr/pgsql-9.3/bin/pg_config
 PG91 = $(shell $(PG_CONFIG) --version | egrep " 8\.| 9\.0" > /dev/null && echo no || echo yes)
 PG92 = $(shell $(PG_CONFIG) --version | egrep " 8\.| 9\.0| 9\.1" > /dev/null && echo no || echo yes)
 
