--- docs/en_US/conf.py.old	2016-11-15 09:56:52.286434350 +0300
+++ docs/en_US/conf.py	2016-11-15 09:57:02.294352958 +0300
@@ -96,7 +96,7 @@
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = 'classic'
+html_theme = 'default'
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
