--- Makefile.old	2014-03-20 13:56:42.774043093 +0200
+++ Makefile	2014-03-20 13:57:17.931792056 +0200
@@ -21,7 +21,7 @@
 	$(CC) $(CFLAGS) $(repmgr_OBJS) $(PG_LIBS) $(LDFLAGS) $(LDFLAGS_EX) $(LIBS) -o repmgr
 
 ifdef USE_PGXS
-PG_CONFIG = pg_config
+PG_CONFIG = /usr/pgsql-9.3/bin/pg_config
 PGXS := $(shell $(PG_CONFIG) --pgxs)
 include $(PGXS)
 else
@@ -36,8 +36,8 @@
 install: install_prog install_ext
 
 install_prog:
-	$(INSTALL_PROGRAM) repmgrd$(X) '$(DESTDIR)$(bindir)'
-	$(INSTALL_PROGRAM) repmgr$(X) '$(DESTDIR)$(bindir)'
+	$(INSTALL_PROGRAM) repmgrd$(X) '$(DESTDIR)$(bindir)/'
+	$(INSTALL_PROGRAM) repmgr$(X) '$(DESTDIR)$(bindir)/'
 
 install_ext:
 	$(MAKE) -C sql install
--- sql/Makefile.old	2014-03-20 14:03:24.685183806 +0200
+++ sql/Makefile	2014-03-20 14:03:36.618098929 +0200
@@ -9,7 +9,7 @@
 OBJS=repmgr_funcs.o
 
 ifdef USE_PGXS
-PG_CONFIG = pg_config
+PG_CONFIG = /usr/pgsql-9.3/bin/pg_config
 PGXS := $(shell $(PG_CONFIG) --pgxs)
 include $(PGXS)
 else
