--- pgcluu.service.old	2019-09-27 16:12:54.622016152 +0100
+++ pgcluu.service	2019-09-27 16:13:29.135226099 +0100
@@ -19,7 +19,7 @@
 ExecStartPre=-/bin/mkdir -p $REPORTDIR
 ExecStartPre=-/bin/chown postgres:www-data $REPORTDIR
 ExecStartPre=-/bin/chmod u=rwX,g=rsX,o= $REPORTDIR
-ExecStart=/usr/local/bin/pgcluu -o $REPORTDIR $STATDIR
+ExecStart=/usr/bin/pgcluu -o $REPORTDIR $STATDIR
 
 [Install]
 WantedBy=multi-user.target
--- pgcluu_collectd.service.old	2019-09-27 16:13:01.330056957 +0100
+++ pgcluu_collectd.service	2019-09-27 16:13:51.095359685 +0100
@@ -20,8 +20,8 @@
 ExecStartPre=-/bin/mkdir -p $STATDIR
 ExecStartPre=-/bin/chown postgres:www-data $STATDIR
 ExecStartPre=-/bin/chmod u=rwX,g=rsX,o= $STATDIR
-ExecStart=/usr/local/bin/pgcluu_collectd --daemonize --rotate-daily --pid-file $PIDFILE $STATDIR
-ExecStop=/usr/local/bin/pgcluu_collectd -k --pid-file $PIDFILE
+ExecStart=/usr/bin/pgcluu_collectd --daemonize --rotate-daily --pid-file $PIDFILE $STATDIR
+ExecStop=/usr/bin/pgcluu_collectd -k --pid-file $PIDFILE
 KillMode=process
 Restart=on-failure
 RestartSec=60s
