--- Makefile.old	2015-02-19 21:21:44.053342788 +0200
+++ Makefile	2015-02-19 21:22:15.302012089 +0200
@@ -21,7 +21,7 @@
 	$(CC) $(CFLAGS) $(repmgr_OBJS) $(PG_LIBS) $(LDFLAGS) $(LDFLAGS_EX) $(LIBS) -o repmgr
 
 ifdef USE_PGXS
-PG_CONFIG = pg_config
+PG_CONFIG = /usr/pgsql-9.0/bin/pg_config
 PGXS := $(shell $(PG_CONFIG) --pgxs)
 include $(PGXS)
 else
--- sql/Makefile.old	2015-02-19 21:21:46.428317653 +0200
+++ sql/Makefile	2015-02-19 21:21:56.855207308 +0200
@@ -9,7 +9,7 @@
 OBJS=repmgr_funcs.o
 
 ifdef USE_PGXS
-PG_CONFIG = pg_config
+PG_CONFIG = /usr/pgsql-9.0/bin/pg_config
 PGXS := $(shell $(PG_CONFIG) --pgxs)
 include $(PGXS)
 else
