--- Makefile.old	2020-11-12 21:38:31.461123649 +0000
+++ Makefile	2020-11-12 21:38:41.582229911 +0000
@@ -2,7 +2,7 @@
 
 PGFILEDESC = "pg_statement_rollback - Automatic rollback at statement level for PostgreSQL"
 
-PG_CONFIG = pg_config
+PG_CONFIG = /usr/pgsql-12/bin/pg_config
 
 # Test that we can install a bgworker, PG >= 9.4
 PG94 = $(shell $(PG_CONFIG) --version | egrep " 8\.| 9\.[0-3]" > /dev/null && echo no || echo yes)
