--- Makefile.old	2015-03-08 19:33:10.953511580 +0200
+++ Makefile	2015-03-08 19:33:30.525296706 +0200
@@ -6,8 +6,8 @@
 DATA         = postcode--$(EXTVERSION).sql
 REGRESS      = parser binary sort random quirks format match partial dps
 REGRESS_OPTS = --load-extension=$(EXTENSION)
-PG_CPPFLAGS  = -Wall -Wpedantic -DEXTVERSION=$(EXTVERSION)
+PG_CPPFLAGS = -std=c99 -Wall -Wpedantic -DEXTVERSION=$(EXTVERSION)
 
-PG_CONFIG = pg_config
+PG_CONFIG = /usr/pgsql-10/bin/pg_config
 PGXS := $(shell $(PG_CONFIG) --pgxs)
 include $(PGXS)
--- postcode.h.old	2015-03-08 19:33:59.989651411 +0200
+++ postcode.h	2015-03-08 19:34:06.584059109 +0200
@@ -14,7 +14,7 @@
 
 #define GET_BITS(var,pos,len)     ((var) >> (pos) & ((1<<(len))-1))
 #define CLR_BITS(var,pos,len)     ((var) &= ~(((1<<(len))-1)<<pos))
-#define SET_BITS(var,pos,len,set) ((var)=(var) &~ (((1<<(len))-1)<<pos) | ((set)<<(pos)))
+#define SET_BITS(var,pos,len,set) ((var) = ((var) &~ (((1<<(len))-1)<<pos)) | ((set)<<(pos)))
 
 #define GET_WALK2(p)       GET_BITS(p,  0, 5)
 #define GET_WALK1(p)       GET_BITS(p,  5, 5)
