--- Makefile.old	2016-03-09 14:58:03.140280274 +0000
+++ Makefile	2016-03-09 14:58:13.673164495 +0000
@@ -14,7 +14,7 @@
 #-----------------------------------------------------------------------------#
 PLV8_VERSION = 1.5.0
 
-PG_CONFIG = pg_config
+PG_CONFIG = /usr/pgsql-9.5/bin/pg_config
 PGXS := $(shell $(PG_CONFIG) --pgxs)
 
 PG_VERSION_NUM := $(shell cat `$(PG_CONFIG) --includedir-server`/pg_config*.h \
