--- src/sample/pgpool.conf.sample.old	2021-12-07 08:27:10.334371396 +0000
+++ src/sample/pgpool.conf.sample	2021-12-07 08:28:13.003465732 +0000
@@ -39,7 +39,7 @@
 #port = 9999
                                    # Port number
                                    # (change requires restart)
-#socket_dir = '/tmp'
+socket_dir = '/run/pgpool-II-10'
                                    # Unix domain socket path
                                    # The Debian package defaults to
                                    # /var/run/postgresql
@@ -292,7 +292,7 @@
 # FILE LOCATIONS
 #------------------------------------------------------------------------------
 
-#pid_file_name = '/var/run/pgpool/pgpool.pid'
+pid_file_name = '/run/pgpool-II-10/pgpool.pid'
                                    # PID file name
                                    # Can be specified as relative to the"
                                    # location of pgpool.conf file or
