--- src/sample/pgpool.conf.sample.old	2022-12-20 13:21:03.390307109 +0000
+++ src/sample/pgpool.conf.sample	2022-12-20 13:21:31.997352399 +0000
@@ -40,7 +40,7 @@
 #port = 9999
                                    # Port number
                                    # (change requires restart)
-#unix_socket_directories = '/tmp'
+unix_socket_directories = '/run/pgpool-II'
                                    # Unix domain socket path(s)
                                    # The Debian package defaults to
                                    # /var/run/postgresql
@@ -341,7 +341,7 @@
 # FILE LOCATIONS
 #------------------------------------------------------------------------------
 
-#pid_file_name = '/var/run/pgpool/pgpool.pid'
+pid_file_name = '/run/pgpool-II/pgpool.pid'
                                    # PID file name
                                    # Can be specified as relative to the"
                                    # location of pgpool.conf file or
