--- pkg/pgadmin3.desktop.old	2015-11-10 11:49:42.592963255 +0000
+++ pkg/pgadmin3.desktop	2015-11-10 12:01:11.559135359 +0000
@@ -1,8 +1,8 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=pgAdmin III
-Exec=/usr/bin/pgadmin3
-Icon=/usr/share/pgadmin3/pgAdmin3.png
+Exec=/usr/pgsql-9.5/bin/pgadmin3
+Icon=/usr/share/icons/hicolor/48x48/apps/pgAdmin3.png
 Type=Application
 Categories=Application;Development;
 MimeType=text/html
