--- bin/pgbackrest.old	2016-08-13 01:12:41.867391713 +0300
+++ bin/pgbackrest	2016-08-13 01:12:48.390335782 +0300
@@ -16,7 +16,6 @@
 use File::Basename qw(dirname);
 use Scalar::Util qw(blessed);
 
-use lib dirname($0) . '/../lib';
 use pgBackRest::Archive;
 use pgBackRest::Common::Exception;
 use pgBackRest::Common::Exit;
--- lib/pgBackRest/Common/Exit.pm.old	2016-08-13 01:13:01.836220490 +0300
+++ lib/pgBackRest/Common/Exit.pm	2016-08-13 01:13:06.963176529 +0300
@@ -12,7 +12,6 @@
 use File::Basename qw(dirname);
 use Scalar::Util qw(blessed);
 
-use lib dirname($0) . '/../lib';
 use pgBackRest::Common::Exception;
 use pgBackRest::Common::Lock;
 use pgBackRest::Common::Log;
--- lib/pgBackRest/Common/Lock.pm.old	2016-08-13 01:13:12.351130330 +0300
+++ lib/pgBackRest/Common/Lock.pm	2016-08-13 01:13:18.644076371 +0300
@@ -12,7 +12,6 @@
 use Fcntl qw(:DEFAULT :flock);
 use File::Basename qw(dirname);
 
-use lib dirname($0) . '/../lib';
 use pgBackRest::Common::Exception;
 use pgBackRest::Common::Log;
 use pgBackRest::Common::String;
--- lib/pgBackRest/Common/Log.pm.old	2016-08-13 01:13:26.590008238 +0300
+++ lib/pgBackRest/Common/Log.pm	2016-08-13 01:13:30.839971797 +0300
@@ -15,7 +15,6 @@
 use Scalar::Util qw(blessed reftype);
 use Time::HiRes qw(gettimeofday usleep);
 
-use lib dirname($0) . '/../lib';
 use pgBackRest::Common::Exception;
 use pgBackRest::Common::String;
 
--- lib/pgBackRest/Common/Wait.pm.old	2016-08-13 01:13:44.733852664 +0300
+++ lib/pgBackRest/Common/Wait.pm	2016-08-13 01:13:49.148972363 +0300
@@ -14,7 +14,6 @@
 use POSIX qw(ceil);
 use Time::HiRes qw(gettimeofday usleep);
 
-use lib dirname($0) . '/../lib';
 use pgBackRest::Common::Log;
 
 ####################################################################################################################################
--- lib/pgBackRest/Config/Config.pm.old	2016-08-13 01:14:00.479875482 +0300
+++ lib/pgBackRest/Config/Config.pm	2016-08-13 01:14:06.790821393 +0300
@@ -14,7 +14,6 @@
 use Getopt::Long qw(GetOptions);
 use Storable qw(dclone);
 
-use lib dirname($0) . '/../lib';
 use pgBackRest::Common::Exception;
 use pgBackRest::Common::Ini;
 use pgBackRest::Common::Log;
--- lib/pgBackRest/Config/ConfigHelp.pm.old	2016-08-13 01:14:13.679762350 +0300
+++ lib/pgBackRest/Config/ConfigHelp.pm	2016-08-13 01:14:20.763701636 +0300
@@ -11,7 +11,6 @@
     our @EXPORT = qw();
 use File::Basename qw(dirname);
 
-use lib dirname($0) . '/../lib';
 use pgBackRest::Common::Exception;
 use pgBackRest::Common::Ini;
 use pgBackRest::Common::Log;
--- lib/pgBackRest/File.pm.old	2016-08-13 01:14:47.598471643 +0300
+++ lib/pgBackRest/File.pm	2016-08-13 01:14:52.536243574 +0300
@@ -17,7 +17,6 @@
 use IO::Handle;
 use Scalar::Util qw(blessed);
 
-use lib dirname($0) . '/../lib';
 use pgBackRest::Common::Exception;
 use pgBackRest::Common::Log;
 use pgBackRest::Common::String;
--- lib/pgBackRest/FileCommon.pm.old	2016-08-13 01:15:10.994257935 +0300
+++ lib/pgBackRest/FileCommon.pm	2016-08-13 01:15:18.647946659 +0300
@@ -16,7 +16,6 @@
 use File::stat;
 use IO::Handle;
 
-use lib dirname($0) . '/../lib';
 use pgBackRest::Common::Exception;
 use pgBackRest::Common::Log;
 
--- lib/pgBackRest/Protocol/CommonMaster.pm.old	2016-08-13 01:15:39.728141824 +0300
+++ lib/pgBackRest/Protocol/CommonMaster.pm	2016-08-13 01:15:45.064096091 +0300
@@ -12,7 +12,6 @@
 use Time::HiRes qw(gettimeofday);
 use Scalar::Util qw(blessed);
 
-use lib dirname($0) . '/../lib';
 use pgBackRest::Common::Exception;
 use pgBackRest::Common::Ini;
 use pgBackRest::Common::Log;
--- lib/pgBackRest/Protocol/CommonMinion.pm.old	2016-08-13 01:15:56.515010213 +0300
+++ lib/pgBackRest/Protocol/CommonMinion.pm	2016-08-13 01:16:01.271003391 +0300
@@ -11,7 +11,6 @@
 use File::Basename qw(dirname);
 use Scalar::Util qw(blessed);
 
-use lib dirname($0) . '/../lib';
 use pgBackRest::Common::Exception;
 use pgBackRest::Common::Ini;
 use pgBackRest::Common::Log;
--- lib/pgBackRest/Protocol/Common.pm.old	2016-08-13 01:16:10.269990481 +0300
+++ lib/pgBackRest/Protocol/Common.pm	2016-08-13 01:16:14.449984484 +0300
@@ -12,7 +12,6 @@
 use Compress::Raw::Zlib qw(WANT_GZIP Z_OK Z_BUF_ERROR Z_STREAM_END);
 use File::Basename qw(dirname);
 
-use lib dirname($0) . '/../lib';
 use pgBackRest::Common::Exception;
 use pgBackRest::Common::Ini;
 use pgBackRest::Common::Log;
--- lib/pgBackRest/Protocol/IO.pm.old	2016-08-13 01:16:25.736968292 +0300
+++ lib/pgBackRest/Protocol/IO.pm	2016-08-13 01:16:29.541962833 +0300
@@ -16,7 +16,6 @@
 use Symbol 'gensym';
 use Time::HiRes qw(gettimeofday);
 
-use lib dirname($0) . '/../lib';
 use pgBackRest::Common::Exception;
 use pgBackRest::Common::Log;
 use pgBackRest::Common::String;
--- lib/pgBackRest/Protocol/RemoteMaster.pm.old	2016-08-13 01:16:41.008946383 +0300
+++ lib/pgBackRest/Protocol/RemoteMaster.pm	2016-08-13 01:16:46.943937869 +0300
@@ -10,7 +10,6 @@
 
 use File::Basename qw(dirname);
 
-use lib dirname($0) . '/../lib';
 use pgBackRest::Common::Log;
 use pgBackRest::Config::Config;
 use pgBackRest::Protocol::CommonMaster;
--- lib/pgBackRest/Protocol/ThreadGroup.pm.old	2016-08-13 01:17:14.222800036 +0300
+++ lib/pgBackRest/Protocol/ThreadGroup.pm	2016-08-13 01:17:19.821133157 +0300
@@ -13,7 +13,6 @@
 use File::Basename;
 use Scalar::Util qw(blessed);
 
-use lib dirname($0) . '/../lib';
 use pgBackRest::Common::Exception;
 use pgBackRest::Common::Log;
 use pgBackRest::Common::Wait;
--- lib/pgBackRest/Protocol/RemoteMinion.pm.old	2016-08-13 01:17:31.261813919 +0300
+++ lib/pgBackRest/Protocol/RemoteMinion.pm	2016-08-13 01:17:36.625133056 +0300
@@ -10,7 +10,6 @@
 
 use File::Basename qw(dirname);
 
-use lib dirname($0) . '/../lib';
 use pgBackRest::Common::Exception;
 use pgBackRest::Common::Log;
 use pgBackRest::Archive;
