--- setup.py.old	2014-10-23 12:18:07.552214856 +0300
+++ setup.py	2014-10-23 12:18:23.933023544 +0300
@@ -49,7 +49,7 @@
 
     distro = platform.linux_distribution()[0]
 
-    if distro == 'CentOS' or distro == 'Red Hat Enterprise Linux Server' or distro == 'Red Hat Enterprise Linux Workstation':
+    if distro == 'Fedora' or distro == 'CentOS' or distro == 'Red Hat Enterprise Linux Server' or distro == 'Red Hat Enterprise Linux Workstation':
         init_file = 'etc/pgbackman_init_rh.sh'
         shutil.copy2(init_file, '/tmp/pgbackman')
 
