diff -up Flask-Babel-0.8/tests/tests.py.orig Flask-Babel-0.8/tests/tests.py
--- Flask-Babel-0.8/tests/tests.py.orig	2014-07-17 09:46:54.630548512 -0300
+++ Flask-Babel-0.8/tests/tests.py	2014-07-17 09:48:25.164551978 -0300
@@ -21,13 +21,13 @@ class DateFormattingTestCase(unittest.Te
         d = datetime(2010, 4, 12, 13, 46)
 
         with app.test_request_context():
-            assert babel.format_datetime(d) == 'Apr 12, 2010 1:46:00 PM'
+            assert babel.format_datetime(d) == 'Apr 12, 2010, 1:46:00 PM'
             assert babel.format_date(d) == 'Apr 12, 2010'
             assert babel.format_time(d) == '1:46:00 PM'
 
         with app.test_request_context():
             app.config['BABEL_DEFAULT_TIMEZONE'] = 'Europe/Vienna'
-            assert babel.format_datetime(d) == 'Apr 12, 2010 3:46:00 PM'
+            assert babel.format_datetime(d) == 'Apr 12, 2010, 3:46:00 PM'
             assert babel.format_date(d) == 'Apr 12, 2010'
             assert babel.format_time(d) == '3:46:00 PM'
 
@@ -43,13 +43,13 @@ class DateFormattingTestCase(unittest.Te
         d = datetime(2010, 4, 12, 13, 46)
 
         with app.test_request_context():
-            assert babel.format_datetime(d) == 'Apr 12, 2010 1:46:00 PM'
+            assert babel.format_datetime(d) == 'Apr 12, 2010, 1:46:00 PM'
             assert babel.format_date(d) == 'Apr 12, 2010'
             assert babel.format_time(d) == '1:46:00 PM'
 
         with app.test_request_context():
             app.config['BABEL_DEFAULT_TIMEZONE'] = 'Europe/Vienna'
-            assert babel.format_datetime(d) == 'Apr 12, 2010 3:46:00 PM'
+            assert babel.format_datetime(d) == 'Apr 12, 2010, 3:46:00 PM'
             assert babel.format_date(d) == 'Apr 12, 2010'
             assert babel.format_time(d) == '3:46:00 PM'
 
@@ -88,7 +88,7 @@ class DateFormattingTestCase(unittest.Te
             return the_timezone
 
         with app.test_request_context():
-            assert babel.format_datetime(d) == 'Apr 12, 2010 1:46:00 PM'
+            assert babel.format_datetime(d) == 'Apr 12, 2010, 1:46:00 PM'
 
         the_locale = 'de_DE'
         the_timezone = 'Europe/Vienna'
@@ -101,10 +101,10 @@ class DateFormattingTestCase(unittest.Te
         b = babel.Babel(app)
         d = datetime(2010, 4, 12, 13, 46)
         with app.test_request_context():
-            assert babel.format_datetime(d) == 'Apr 12, 2010 1:46:00 PM'
+            assert babel.format_datetime(d) == 'Apr 12, 2010, 1:46:00 PM'
             app.config['BABEL_DEFAULT_TIMEZONE'] = 'Europe/Vienna'
             babel.refresh()
-            assert babel.format_datetime(d) == 'Apr 12, 2010 3:46:00 PM'
+            assert babel.format_datetime(d) == 'Apr 12, 2010, 3:46:00 PM'
 
 
 class NumberFormattingTestCase(unittest.TestCase):
