diff --git a/tox.ini b/tox.ini
index ffb40c5..1211611 100644
--- a/tox.ini
+++ b/tox.ini
@@ -9,10 +9,10 @@ commands =
     python -c "import os, sys; sys.platform == 'win32' and os.system('easy_install pillow')"
     py.test --cov-report term-missing --cov-report xml --cov {[general]name} --cov-config tox.ini {posargs:tests}
 deps =
-    colorama==0.3.7
-    colorclass==2.2.0
-    pytest-cov==2.4.0
-    termcolor==1.1.0
+    colorama>=0.3.7
+    colorclass>=2.2.0
+    pytest-cov>=2.4.0
+    termcolor>=1.1.0
 passenv =
     WINDIR
 setenv =
