--- setup.py.old	2017-01-25 01:09:29.547597152 +0300
+++ setup.py	2017-01-25 01:09:47.958423593 +0300
@@ -68,7 +68,7 @@
 
 def pg_config(s):
     """Retrieve information about installed version of PostgreSQL."""
-    f = os.popen('pg_config --%s' % s)
+    f = os.popen('/usr/pgsql-9.5/bin/pg_config --%s' % s)
     d = f.readline().strip()
     if f.close() is not None:
         raise Exception("pg_config tool is not available.")
