--- Makefile.old	2020-11-17 00:47:04.277036326 +0000
+++ Makefile	2020-11-17 00:47:14.901148761 +0000
@@ -4,7 +4,7 @@
 
 PGFILEDESC = "pgtt - Global Temporary Tables for PostgreSQL"
 
-PG_CONFIG = pg_config
+PG_CONFIG = /usr/pgsql-12/bin/pg_config
 
 PGVEROK = $(shell $(PG_CONFIG) --version | egrep " (9.[456]|1[0123])" > /dev/null && echo yes || echo no)
 PG_CPPFLAGS = -I$(libpq_srcdir) -Wno-uninitialized
