--- mk/catsql.py.old	2020-02-18 13:11:44.798384162 +0300
+++ mk/catsql.py	2020-02-18 13:11:51.518427278 +0300
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python3
 
 """Prints out SQL files with psql command execution.
 
--- mk/grantfu.py.old	2020-02-18 13:11:41.661364033 +0300
+++ mk/grantfu.py	2020-02-18 13:11:50.037417777 +0300
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python3
 
 # GrantFu - GRANT/REVOKE generator for Postgres
 # 
