--- Makefile.old	2018-12-21 14:25:57.527805704 +0300
+++ Makefile	2018-12-21 14:26:09.345874980 +0300
@@ -6,7 +6,7 @@
 DATA = orafce--3.7.sql orafce--3.2--3.3.sql orafce--3.3--3.4.sql orafce--3.4--3.5.sql orafce--3.5--3.6.sql orafce--3.6--3.7.sql
 DOCS = README.asciidoc COPYRIGHT.orafce INSTALL.orafce
 
-PG_CONFIG ?= pg_config
+PG_CONFIG ?= /usr/pgsql-9.4/bin/pg_config
 
 # make "all" the default target
 all:
@@ -25,7 +25,7 @@
 include $(top_builddir)/src/Makefile.global
 include $(top_srcdir)/contrib/contrib-global.mk
 else
-PG_CONFIG = pg_config
+PG_CONFIG = /usr/pgsql-9.4/bin/pg_config
 PGXS := $(shell $(PG_CONFIG) --pgxs)
 include $(PGXS)
 endif
