--- Makefile.old	2019-03-15 14:49:09.889104506 +0000
+++ Makefile	2019-03-15 14:49:38.686657572 +0000
@@ -8,7 +8,7 @@
 
 EXTENSION = pg_store_plans
 
-PG_VERSION := $(shell pg_config --version | sed "s/^PostgreSQL //" | sed "s/\.[0-9]*$$//")
+PG_VERSION := $(shell /usr/pgsql-10/bin/pg_config --version | sed "s/^PostgreSQL //" | sed "s/\.[0-9]*$$//")
 
 DATA = pg_store_plans--1.3.sql
 ifeq ($(PG_VERSION),10)
@@ -21,7 +21,7 @@
 REGRESS = convert store
 REGRESS_OPTS = --temp-config=regress.conf
 ifdef USE_PGXS
-PG_CONFIG = pg_config
+PG_CONFIG = /usr/pgsql-10/bin/pg_config
 PGXS := $(shell $(PG_CONFIG) --pgxs)
 include $(PGXS)
 else
