--- Makefile.old	2015-09-25 23:29:07.746726988 +0300
+++ Makefile	2015-09-25 23:29:16.445641259 +0300
@@ -32,7 +32,7 @@
 
 DATA = sql/$(EXTENSION)--$(EXTVERSION).sql
 
-PG_CONFIG    = pg_config
+PG_CONFIG    = /usr/pgsql-9.4/bin/pg_config
 
 # modify these variables to point to FreeTDS, if needed
 SHLIB_LINK := -lsybdb
@@ -50,4 +50,4 @@
 EXTRA_CLEAN = sql/$(EXTENSION)--$(EXTVERSION).sql README.${EXTENSION}.md
 
 PGXS := $(shell $(PG_CONFIG) --pgxs)
-include $(PGXS)
\ No newline at end of file
+include $(PGXS)
