--- configure.ac.old	2020-03-12 23:30:44.971928895 +0000
+++ configure.ac	2020-03-12 23:30:59.287928895 +0000
@@ -360,7 +360,4 @@
 
 AC_CONFIG_FILES([proj.pc])
 
-dnl Ignore WARNING:  'proj.pc.in' seems to ignore the --datarootdir setting
-AC_DEFUN([AC_DATAROOTDIR_CHECKED])
-
 AC_OUTPUT
--- proj.pc.in.old	2020-03-12 23:31:16.403928895 +0000
+++ proj.pc.in	2020-03-12 23:31:24.523928895 +0000
@@ -2,6 +2,7 @@
 exec_prefix=@exec_prefix@
 libdir=@libdir@
 includedir=@includedir@
+datarootdir=@datarootdir@
 datadir=@datadir@/@PACKAGE@
 
 Name: PROJ
