--- Makefile.contrib.old	2014-06-25 23:20:42.671716269 +0300
+++ Makefile.contrib	2014-06-25 23:20:54.528679173 +0300
@@ -4,7 +4,7 @@
 DOCS = README.pg_filedump
 
 ifdef USE_PGXS
-PG_CONFIG = pg_config
+PG_CONFIG = /usr/pgsql-9.3/bin/pg_config
 PGXS := $(shell $(PG_CONFIG) --pgxs)
 include $(PGXS)
 else
--- Makefile.old	2014-06-25 23:23:46.503368731 +0300
+++ Makefile	2014-06-25 23:24:42.159950174 +0300
@@ -9,7 +9,7 @@
 # If working with a PG source directory, point PGSQL_INCLUDE_DIR to its
 # src/include subdirectory.  If working with an installed tree, point to
 # the server include subdirectory, eg /usr/local/include/postgresql/server
-PGSQL_INCLUDE_DIR=../../pgsql/src/include
+PGSQL_INCLUDE_DIR=/usr/pgsql-9.3/include/server
 
 
 DISTFILES= README.pg_filedump Makefile Makefile.contrib \
