--- Makefile.old	2016-06-08 14:14:10.461757733 +0300
+++ Makefile	2016-06-08 14:17:44.699703827 +0300
@@ -6,7 +6,7 @@
 DATA = orafce--3.3.sql orafce--3.2--3.3.sql
 DOCS = README.asciidoc COPYRIGHT.orafce INSTALL.orafce
 
-PG_CONFIG ?= pg_config
+PG_CONFIG ?= /usr/pgsql-9.3/bin/pg_config
 
 # version as a number, e.g. 9.1.4 -> 901
 VERSION := $(shell $(PG_CONFIG) --version | awk '{print $$2}')
@@ -29,7 +29,7 @@
 include $(top_builddir)/src/Makefile.global
 include $(top_srcdir)/contrib/contrib-global.mk
 else
-PG_CONFIG = pg_config
+PG_CONFIG = /usr/pgsql-9.3/bin/pg_config
 PGXS := $(shell $(PG_CONFIG) --pgxs)
 include $(PGXS)
 endif
