--- cmake/FindPostgreSQL.cmake.old	2015-12-30 08:49:28.396080697 +0200
+++ cmake/FindPostgreSQL.cmake	2015-12-30 08:50:27.700426487 +0200
@@ -34,6 +34,7 @@
         # Checking POSTGRESQL_PG_CONFIG 
         find_program(POSTGRESQL_PG_CONFIG NAMES pg_config
             PATHS
+	    /usr/pgsql-9.5/bin/
             /usr/lib/postgresql/*/bin/
             )
     endif(NOT "${POSTGRESQL_BIN}" STREQUAL "")
@@ -58,12 +59,14 @@
 
 #    find_program(POSTGRESQL_EXECUTABLE NAMES postgres
 #        PATHS
+#	 /usr/pgsql-9.5/bin/
 #        /usr/lib/postgresql/*/bin/
 #        )
 #    message(STATUS "POSTGRESQL_EXECUTABLE is " ${POSTGRESQL_EXECUTABLE})
 
 #    find_program(POSTGRESQL_PG_CONFIG NAMES pg_config
 #        PATHS
+#	 /usr/pgsql-9.5/bin/
 #        /usr/lib/postgresql/*/bin/
 #        )
 #    message(STATUS "POSTGRESQL_PG_CONFIG is " ${POSTGRESQL_PG_CONFIG})
@@ -87,6 +90,7 @@
     find_path(POSTGRESQL_INCLUDE_DIR postgres.h
         ${T_POSTGRESQL_INCLUDE_DIR}
 
+	/usr/pgsql-9.5/include/server
         /usr/include/server
         /usr/include/pgsql/server
         /usr/local/include/pgsql/server
