--- Makefile.old	2016-02-19 10:43:52.256394460 +0200
+++ Makefile	2016-02-19 10:44:06.668260656 +0200
@@ -6,7 +6,7 @@
 DATA = orafce--3.2.sql
 DOCS = README.asciidoc COPYRIGHT.orafce INSTALL.orafce
 
-PG_CONFIG ?= pg_config
+PG_CONFIG ?= /usr/pgsql-9.6/bin/pg_config
 
 # version as a number, e.g. 9.1.4 -> 901
 VERSION := $(shell $(PG_CONFIG) --version | awk '{print $$2}')
@@ -29,7 +29,7 @@
 include $(top_builddir)/src/Makefile.global
 include $(top_srcdir)/contrib/contrib-global.mk
 else
-PG_CONFIG = pg_config
+PG_CONFIG = /usr/pgsql-9.6/bin/pg_config
 PGXS := $(shell $(PG_CONFIG) --pgxs)
 include $(PGXS)
 endif
