--- Makefile.old	2015-03-16 08:37:13.741552309 +0200
+++ Makefile	2015-03-16 08:37:25.571404542 +0200
@@ -9,7 +9,7 @@
 TESTS		= $(wildcard test/sql/*.sql)
 REGRESS		= $(patsubst test/sql/%.sql,%,$(TESTS))
 REGRESS_OPTS= --inputdir=test
-PG_CONFIG	= pg_config
+PG_CONFIG	= /usr/pgsql-9.6/bin/pg_config
 LIBJSON		= libjson-0.8
 # use module big instead:
 #MODULES		= $(patsubst %.c,%,$(wildcard src/*.c))
