--- Makefile.old	2016-04-18 23:26:00.529028291 +0300
+++ Makefile	2016-04-18 23:26:07.084961532 +0300
@@ -30,7 +30,7 @@
 $(repmgrd_OBJS): $(HEADERS)
 
 ifdef USE_PGXS
-PG_CONFIG = pg_config
+PG_CONFIG = /usr/pgsql-9.3/bin/pg_config
 PGXS := $(shell $(PG_CONFIG) --pgxs)
 include $(PGXS)
 else
--- sql/Makefile.old	2016-04-18 23:26:10.500926747 +0300
+++ sql/Makefile	2016-04-18 23:26:15.570875120 +0300
@@ -10,7 +10,7 @@
 OBJS=repmgr_funcs.o
 
 ifdef USE_PGXS
-PG_CONFIG = pg_config
+PG_CONFIG = /usr/pgsql-9.3/bin/pg_config
 PGXS := $(shell $(PG_CONFIG) --pgxs)
 include $(PGXS)
 else
