--- repmgr.conf.sample.old	2017-01-07 16:47:17.116581773 +0300
+++ repmgr.conf.sample	2017-01-07 16:49:17.986512563 +0300
@@ -78,7 +78,7 @@
 
 # stderr can be redirected to an arbitrary file:
 #
-#logfile='/var/log/repmgr/repmgr.log'
+logfile='/var/log/repmgr/repmgr-9.4.log
 
 # By default only repmgrd log output will be written to a file,
 # if defined in "logfile"
@@ -112,6 +112,8 @@
 # path to PostgreSQL binary directory (location of pg_ctl, pg_basebackup etc.)
 # (if not provided, defaults to system $PATH)
 #pg_bindir=/usr/bin/
+pg_bindir=/usr/pgsql-9.4/bin/
+
 #
 # Debian/Ubuntu users: you will probably need to set this to the directory
 # where `pg_ctl` is located, e.g. /usr/lib/postgresql/9.5/bin/
@@ -135,11 +137,11 @@
 #       /usr/bin/systemctl start postgresql-9.5, \
 #       /usr/bin/systemctl restart postgresql-9.5
 #
-# service_start_command = systemctl start postgresql-9.5
-# service_stop_command = systemctl stop postgresql-9.5
-# service_restart_command = systemctl restart postgresql-9.5
-# service_reload_command = pg_ctlcluster 9.5 main reload
-# service_promote_command = pg_ctlcluster 9.5 main promote
+# service_start_command = systemctl start postgresql-9.4
+# service_stop_command = systemctl stop postgresql-9.4
+# service_restart_command = systemctl restart postgresql-9.4
+# service_reload_command = systemctl reload postgresql-9.4
+# service_promote_command = /usr/pgsql-9.4/bin/pg_ctl -D /var/lib/pgsql/9.4/data-standby promote
 
 # external command options
 
@@ -198,8 +200,8 @@
                     # a value of zero or less prevents the node being promoted to primary
                     # (default: 100)
 
-#promote_command='repmgr standby promote -f /path/to/repmgr.conf'
-#follow_command='repmgr standby follow -f /path/to/repmgr.conf -W'
+promote_command='/usr/pgsql-9.4/bin/repmgr standby promote -f /etc/repmgr/9.4/repmgr.conf'
+follow_command='/usr/pgsql-9.4/bin/repmgr standby follow -f /etc/repmgr/9.4/repmgr.conf'
 
 # change wait time for primary; before we bail out and exit when the primary
 # disappears, we wait 'reconnect_attempts' * 'retry_promote_interval_secs'
