--- configure.ac.old	2015-11-24 08:40:37.819406506 +0200
+++ configure.ac	2015-11-24 08:41:01.862146314 +0200
@@ -374,8 +374,6 @@
 
 fi
 
-AC_CHECK_SIZEOF(long)
-
 AC_MSG_CHECKING(for F_SETFD in fcntl.h)
 AC_EGREP_CPP(yes,
 [
@@ -552,12 +550,7 @@
 #include <sys/select.h>
 #endif])
 
-AC_CHECK_SIZEOF(long long)
-AC_CHECK_SIZEOF(long)
-AC_CHECK_SIZEOF(int)
-AC_CHECK_SIZEOF(short)
 AC_CHECK_SIZEOF(size_t)
-AC_CHECK_SIZEOF(void *)
 AC_CHECK_SIZEOF(off_t)
 
 AC_CHECK_TYPES([struct in6_addr, struct sockaddr_in6, sa_family_t, struct addrinfo, struct sockaddr_storage], , ,
