--- Makefile.old	2016-03-09 14:49:54.569649844 +0000
+++ Makefile	2016-03-09 14:50:11.505463908 +0000
@@ -5,7 +5,7 @@
 DOCS         = $(wildcard doc/*.md)
 SCRIPTS      = $(wildcard tools/*.p[lm]) $(wildcard tools/*.sh)
 
-PG_CONFIG    = pg_config
+PG_CONFIG    = /usr/pgsql-9.5/bin/pg_config
 DATA         = $(wildcard sql/$(EXTENSION)--*--*.sql) sql/$(EXTENSION)--$(EXTVERSION).sql
 
 PGXS := $(shell $(PG_CONFIG) --pgxs)
